<html xmlns="http://www.w3.org/1999/xhtml"><head> <style type="text/css">
         	     		 
         	    a                          {background : #ffffff; }
         	    article                    {border-style : dotted; border-width : 2px; }
         		 	
         		 	div                        {background : #ffffcc;}
         		 	div.abstract-title         {font-weight : bold ; font-size : 16pt;}
         		 	div.ack                    {border-style : solid ; border-color : red; margin
         : 2em; }
         		 	div.article-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.boxed-text             {margin : 5em; border-style : solid;}
         		 	div.contrib-group          {margin : 1em; }
         		 	div.fig                    {border-style : solid; border-width : 2px; margin :
         2em; }
         		 	div.funding                {font-weight : bold ; font-size : 16pt;}
         		 	div.given-names            {font-style : italic;}
         		 	div.intro                  {border-style : inset; margin : 5px;}
         		 	div.introduction           {border-style : inset; margin : 5px;}
         		 	div.journal-meta           {border-style : solid; border-width : 1 pt; margin
         1em;}
         		 	div.journal-title-group    {background : #ffeeee;}
         		 	div.article-title          {font-weight : bold ; font-size : 18pt;}
         		 	div.kwd                    {font-style : italic}
         		 	div.meta-name              {font-weight : bold ; font-size : 16pt;}
         		 	div.name                   {font-weight : bold;}
         		 	div.alt-title              {font-style : italic; font-size : 12px;}
         		 	
         		 	div.sec                    {border : 2px; margin 5px; padding 2px;}
         		 	div.title                  {font-family : courier; font-weight : bold;
         		 	                            font-size : 16pt; margin : 5px;}
         		 	
         		 	div.materials_methods      {border-style : double; margin : 5px; }
         		 	div.methods                {border-style : double; margin : 5px; }
         		 	
         		 	div.results                {border-style : solid; margin : 5px;}
         		 	div.background             {border-style : dotted; margin : 5px;}
         		 	
         		 	div.discussion             {border-style : groove; margin : 5px;}
         		 	div.conclusion             {border-style : ridge; margin : 5px;}
         		 	div.conclusions             {border-style : ridge; margin : 5px;}
         		 	div.supplementary-material {border-style : inset; margin : 5px;}
         		 	div.abbreviations          {border-style : double; border-color : red; margin
         : 5px;}
         		 	div.competinginterests     {border-style : double; border-color : blue; margin
         : 5px;}
         		 	div.acknowledgements       {border-style : double; border-color : green; margin
         : 5px;}
         		 	div.authors_contributions   {border-style : double; border-color : purple; margin
         : 5px;}
         		 	
         		 	div.publisher              {border-style : outset; margin : 5px;}
         		 	div.fn-type-conflict       {background : #f88; }
         		 	div.fn-type-con            {background : #ddf; }
         		 	div.fn-type-other          {background : #ddd; }
         		 	
         		 	div.unknown                {background : #ffd;
         									 	  border-style : solid;
         									 	  border-width : 1px;
         									 	  padding : 2 px;}
         		 	  
         	    table                      {background : #ffffdd;}
         		 	tr                         {background : #ddddff; padding : 1px;}
         		 	
         		 	span                       {background : #ffcccc;}
         		 	
         		 	span.citation-author       {font-family : helvetica; background : #ffeeee;}
         		 	span.collab                {background : #ddffff; }
         		 	span.comment               {font-family : courier; font-size : 6px; background
         : #ffaaff;}
         		 	span.contrib               {background : #ffffff;}
         		 	span.corresp               {background : #ddffdd; }
         		 	span.doi                   {background : #ffffff;}
         		 	span.email                 {font-family : courier; }
         		 	span.etal                  {font-style : italic;}
         		 	span.fpage                 {font-family : courier;}
         		 	span.given-names           {background : #ffffff;}
         		 	span.iso-abbrev            {background : #ffffff;}
         		 	span.issn-epub             {background : #ffffff;}
         		 	span.issn-ppub             {background : #ffffff;}
         		 	span.journal-title         {background : #ffffff;}
         		 	span.lpage                 {font-family : courier;}
         		 	span.mixed-article-title   {font-style : italic ;}
         		 	span.nlm-ta                {background : #ffffff;}
         		 	span.pmc                   {background : #ffffff;}
         		 	span.pmcid                 {background : #ffffff;}
         		 	span.pmid                  {background : #ffffff;}
         		 	span.publisher             {background : #ffffff;}
         		 	span.publisher-id          {background : #ffffff;}
         		 	span.publisher-name        {background : #ffffff;}
         		 	span.source                {background : #ffffff;}
         		 	span.subject               {background : #ffffff;}
         		 	span.surname               {background : #ffffff;}
         		 	span.volume                {font-family : courier; font-weight : bold;}
         		 	span.year                  {font-family : courier ; font-style : italic;}
         			
      </style> </head> <body> <div class="front" title="front"> <div class="journal-meta" tagx="journal-meta" title="journal-meta"> <span class="nlm-ta" title="nlm-ta">Microorganisms</span> <span class="iso-abbrev" title="iso-abbrev">Microorganisms</span> <span class="publisher-id" title="publisher-id">microorganisms</span> <div class="journal-title-group" tagx="journal-title-group" title="journal-title-group"> <span class="journal-title" tagx="journal-title" title="journal-title">Microorganisms</span> </div> <span class="issn-epub" tagx="issn" title="issn-epub">2076-2607</span> <div class="publisher" tagx="publisher" title="publisher"> <span class="publisher-name" tagx="publisher-name" title="publisher-name">MDPI</span> </div> </div> <div class="article-meta" tagx="article-meta" title="article-meta"> <span class="pmcid" title="pmcid"> pmcid: <a href="http://www.ncbi.nlm.nih.gov/pubmed/6560395">6560395</a> </span> <span class="doi" title="doi"> doi: <a href="https://dx.doi.org/10.3390/microorganisms7050115">10.3390/microorganisms7050115</a> </span> <span class="publisher-id" title="publisher-id">microorganisms-07-00115</span> <div class="article-categories" title="article-categories">
: <span class="subject" title="subject">Article</span> </div> <div class="title-group"> <div class="article-title" title="article-title">
Inhibiting Methanogenesis in Rumen Batch Cultures Did Not Increase the Recovery of Metabolic Hydrogen in Microbial Amino Acids </div> </div> <div class="contrib-group" title="contrib-group"> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0002-5422-5462</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ungerfeld</span> <span class="given-names" tagx="given-names" title="given-names">Emilio M.</span> </span> <a href="#af1-microorganisms-07-00115">1</a> <a href="#c1-microorganisms-07-00115">*</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aedo</span> <span class="given-names" tagx="given-names" title="given-names">M. Fernanda</span> </span> <a href="#af1-microorganisms-07-00115">1</a> </span> <span class="contrib" title="contrib"> <span class="contrib-id" tagx="contrib-id" title="contrib-id">https://orcid.org/0000-0003-1009-5405</span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martínez</span> <span class="given-names" tagx="given-names" title="given-names">Emilio D.</span> </span> <a href="#af2-microorganisms-07-00115">2</a> </span> <span class="contrib" title="contrib"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Saldivia</span> <span class="given-names" tagx="given-names" title="given-names">Marcelo</span> </span> <a href="#af2-microorganisms-07-00115">2</a> </span> </div> <span class="citation_author_institution" id="af1-microorganisms-07-00115">[1], <span class="email" tagx="email" title="email">aedo.enriquez13@gmail.com</span> </span> <span class="citation_author_institution" id="af2-microorganisms-07-00115">[2], <span class="email" tagx="email" title="email">emiliomartinez@uach.cl</span> <span class="email" tagx="email" title="email">marcelosaldiviamv@gmail.com</span> </span> <div class="author-notes" title="author-notes"> <div class="corresp" title="corresp"> <span class="label" tagx="label" title="label">*</span>Correspondence: <span class="email" tagx="email" title="email">emilio.ungerfeld@inia.cl</span>; Tel.: +56-45-2297296 </div> </div> <span class="pub-date-epub" title="pub-date-epub">epub: <span>2019-4-4</span> </span> <span class="pub-date-collection" title="pub-date-collection">collection: <span>2019-5</span> </span> <span class="volume" tagx="volume" title="volume">7</span> <span class="issue" tagx="issue" title="issue">5</span> <span class="elocation-id" tagx="elocation-id" title="elocation-id">115</span> <span class="history" title="history"> <span class="received" title="received">received: 2019-3-29</span> <span class="accepted" title="accepted">accepted: 2019-4-23</span> </span> <div class="permissions"> <span class="copyright" title="copyright">(C) , <span class="copyright-year" tagx="copyright-year" title="copyright-year">2019</span> </span> <span class="license" title="license"> <span class="license-p" title="license-p">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license ( <a href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</a>). </span> </span> </div> <div class="abstract" title="abstract"> <div class="abstract-title" title="abstract-title">
Abstract</div> <p>There is an interest in controlling rumen methanogenesis as an opportunity to both decrease the emissions of greenhouse gases and improve the energy efficiency of rumen fermentation. However, the effects of inhibiting rumen methanogenesis on fermentation are incompletely understood even in in vitro rumen cultures, as the recovery of metabolic hydrogen ([H]) in the main fermentation products consistently decreases with methanogenesis inhibition, evidencing the existence of unaccounted [H] sinks. We hypothesized that inhibiting methanogenesis in rumen batch cultures would redirect [H] towards microbial amino acids (AA) biosynthesis as an alternative [H] sink to methane (CH <sub>4</sub>). The objective of this experiment was to evaluate the effects of eight inhibitors of methanogenesis on digestion, fermentation and the production of microbial biomass and AA in rumen batch cultures growing on cellulose. Changes in the microbial community composition were also studied using denaturing gradient gel electrophoresis (DGGE). Inhibiting methanogenesis did not cause consistent changes in fermentation or the profile of AA, although the effects caused by the different inhibitors generally associated with the changes in the microbial community that they induced. Under the conditions of this experiment, inhibiting methanogenesis did not increase the importance of microbial AA synthesis as a [H] sink. </p> </div> <div class="kwd-group"> <div class="kwd" title="kwd">
rumen</div> <div class="kwd" title="kwd">
fermentation</div> <div class="kwd" title="kwd">
methane</div> <div class="kwd" title="kwd">
inhibition</div> <div class="kwd" title="kwd">
metabolic hydrogen</div> <div class="kwd" title="kwd">
amino acids</div> <div class="kwd" title="kwd">
microbial community composition</div> </div> </div> </div> <div class="body" title="body"> <div class="intro" title="intro"> <div class="title" tagx="title" title="title">
1. Introduction</div> <p>Ruminants can transform roughages and non-protein nitrogen (N) otherwise unavailable to humans into meat, milk, wool, and traction. Essential to this nutritional flexibility is the presence of a complex microbial community in the rumen, which is able to degrade structural carbohydrates and synthesize amino acids (AA) by incorporating ammonia into carbon chains. In the fermentation of monosaccharides released from the degradation of complex carbohydrates in the rumen, metabolic hydrogen ([H]) is generated in redox reactions in bacterial, protozoal, and fungal cells, reducing intracellular cofactors. For fermentation to continue reduced cofactors need to be re-oxidized, which to an important extent occurs through the formation of dihydrogen (H <sub>2</sub>) [ <a href="#B1-microorganisms-07-00115">1</a>, <a href="#B2-microorganisms-07-00115">2</a>]. Dihydrogen is a central intercellular intermediate in rumen fermentation, with hydrogenases-encoding genes being widespread in rumen bacteria and Archaea [ <a href="#B3-microorganisms-07-00115">3</a>]. Most H <sub>2</sub>, along with other [H] donors such as formate, methanol, and methylamines is incorporated into methane (CH <sub>4</sub>) [ <a href="#B2-microorganisms-07-00115">2</a>], which is the main electron sink in rumen fermentation. </p> <p>Because CH <sub>4</sub> formed in the rumen and released to the atmosphere constitutes a greenhouse gas [ <a href="#B4-microorganisms-07-00115">4</a>] and also represents energy that is not incorporated into fermentation products that the animal can absorb and utilize [ <a href="#B5-microorganisms-07-00115">5</a>], controlling the emissions of CH <sub>4</sub> from ruminants can be regarded as an opportunity to simultaneously ameliorate climate change and improve animal productivity. </p> <p>Various strategies to control CH <sub>4</sub> formation in the rumen are currently being investigated. Rumen methanogenesis can be strongly inhibited by various chemical compounds [ <a href="#B6-microorganisms-07-00115">6</a>] and oils such as linseed oil [ <a href="#B7-microorganisms-07-00115">7</a>]. Whereas some of these additives and ingredients can inhibit methane production effectively, benefits in production have been inconsistent despite the theoretical gain of energy not lost as CH <sub>4</sub> [ <a href="#B8-microorganisms-07-00115">8</a>]. In vitro evidence from multiple experiments meta-analyzed has shown some undesirable and incompletely understood consequences of inhibiting methanogenesis, such as a decrease in total enthalpy output in volatile fatty acids (VFA) and a consistent decrease in the recovery of reducing equivalents pairs ([2H]) recovered in the main fermentation products [ <a href="#B9-microorganisms-07-00115">9</a>]. It would be important to gain a thorough understanding of the changes occurring in [H] sinks when methanogenesis is inhibited in rumen fermentation. </p> <p>It has been proposed that some of the unaccounted [2H] incorporated into the main fermentation products when methanogenesis is inhibited may be explained by increases in microbial biomass production, along with changes in its composition [ <a href="#B9-microorganisms-07-00115">9</a>]. Previously, Hino and Russell [ <a href="#B10-microorganisms-07-00115">10</a>] observed that inhibiting methanogenesis decreased deamination of AA in trypticase. Amination involves incorporation of one mol of [2H] per mol of ammonia, whereas deamination involves the release of [2H] [ <a href="#B11-microorganisms-07-00115">11</a>]. Thus, part of the unaccounted [2H] resulting from methanogenesis inhibition could perhaps be explained by greater AA synthesis and lesser deamination. The main objective of this experiment was to study the changes in fermentation products and microbial AA as a consequence of inhibiting methanogenesis in rumen batch cultures using a range of chemical inhibitors and linseed oil. Our hypothesis was that changes in the synthesis of microbial AA could partly explain the decrease in [2H] recovery in main fermentation products when methanogenesis is inhibited. We also conducted a denaturing gradient gel electrophoresis (DGGE) to provide insights about how changes observed in fermentation and microbial growth associated with changes in the microbial community composition. Under the conditions of this experiment, uptake of [2H] for microbial AA biosynthesis did not explain the consistent decline occurring in [2H] recovery when methanogenesis is inhibited. </p> </div> <div class="2.materialsandmethods" title="sec"> <div class="title" tagx="title" title="title">
2. Materials and Methods</div> <p>All procedures with animals employed in Project Fondecyt 1160764 were approved by Comité de Ética para el Uso de Animales en Investigación, Instituto de Investigaciones Agropecuarias INIA, Approval number 02/2016, on June 7 2016. </p> <div class="2.1.treatmentsandincubations" title="sec"> <div class="title" tagx="title" title="title">
2.1. Treatments and Incubations</div> <p>Approximately 0.5 L of rumen contents were sampled from the center of the rumen of each of two ruminally-cannulated non-pregnant, non-lactating, Holstein cows. Cows were fed ryegrass ( <i>Lolium multiflorum</i>) hay [5.8% crude protein, 59.4% neutral detergent fiber (NDF), 6.0% total ashes, dry matter (DM) basis] once per day in the morning. Rumen contents were sampled before feeding around 10 am, and filtered through two layers of synthetic cloth. The fluid fraction from both cows was pooled, and the pooled fluid as well as the solids were transported to the laboratory in separate insulated containers. </p> <p>One hundred milliliters of pooled rumen fluid were delivered under oxygen (O <sub>2</sub>)-free carbon dioxide (CO <sub>2</sub>) into a 500 mL Erlenmeyer flask, and approximately 50 mL of rumen solids from both cows were added. The resulting mixture of fluid and solids was subjected to an eggbeater at low speed under O <sub>2</sub>-free CO <sub>2</sub> for 1 min (composed by bursts of approximately 3 s followed by 2 s breaks) to detach microorganisms adhered to solid particles. After egg beating, the mixture of fluid and solids was filtered through two layers of synthetic cloth under O <sub>2</sub>-free CO <sub>2</sub> to obtain the rumen inoculum. </p> <p>Subsequently, 130 µL of the resulting rumen inoculum was delivered under O <sub>2</sub>-free CO <sub>2</sub> into 250 mL serum bottles containing 130 mL of the Goering and van Soest [ <a href="#B12-microorganisms-07-00115">12</a>] medium including its reducing solution and with the following modifications: (i) Trypticase was not included; (ii) addition of 0.5 g/L yeast extract (Sigma-Aldrich 70161, St Louis, MO, USA); (iii) addition of 10 mL/L of Pfennigs mineral solution [ <a href="#B13-microorganisms-07-00115">13</a>]; (iv) 10 mL/L of VFA solution [ <a href="#B14-microorganisms-07-00115">14</a>]. Each bottle contained 800 mg cellulose (Sigma C6288, St Louis, MO, USA) of which its exact mass was recorded. Each bottle also contained one of the following: (i) 1 mL distilled water (Control); (ii) 235 µL of a 30 mM 2-bromoethanesulfonate (BES; Sigma-Aldrich 137502) solution; (iii) 1.3 mg anthraquinone (AQ; Sigma-Aldrich A90004) and 1 mL distilled water; (iv) 209 µL of a 0.05% ( <i>v</i>/ <i>v</i>) chloroform (CL; Merck 1.02445.2500, Darmstadt, Germany) solution and 791 µL distilled water; (v) 256 µL of a 0.05% ( <i>v</i>/ <i>v</i>) bromotrichloromethane (BTCM; Merck 8.01986.0100) solution and 744 µL distilled water; (vi) 40 µL of propynoic acid (propiolic acid; PA; Sigma-Aldrich P51400) and 1 mL distilled water; (vii) 75 µL of ethyl 2-butynoate (E2B; Sigma-Aldrich 425117) and 1 mL distilled water; viii) 163 µL linseed oil (LO; Nutra Andes Ltda., Valparaíso, Chile) and 1 mL distilled water; or (ix) 1 mL of a 1.3 M sodium nitrate (SN; Sigma-Aldrich S5506) solution. Final concentrations were chosen to target at least a 50% CH <sub>4</sub> decrease relative to the control treatment and were 50 µM BES [ <a href="#B15-microorganisms-07-00115">15</a>, <a href="#B16-microorganisms-07-00115">16</a>], 10 ppm AQ [ <a href="#B17-microorganisms-07-00115">17</a>], 1.0 µM CL [ <a href="#B18-microorganisms-07-00115">18</a>], 1.0 µM BTCM (concentration chosen equal to CL), 5 mM PA [ <a href="#B19-microorganisms-07-00115">19</a>], 5 mM E2B [ <a href="#B19-microorganisms-07-00115">19</a>], 1.25 mL/L LO, and 10 mM SN [ <a href="#B20-microorganisms-07-00115">20</a>]. Bottles were immediately sealed and placed in a shaking water bath at 39 °C and 60 cycles/min. The initial gas pressure above ambient was determined using a pressure transducer (Sper Scientific 840065, Scottsdale, AZ, USA) after allowing the bottles to warm up for 10 min. </p> </div> <div class="methods" title="methods"> <div class="title" tagx="title" title="title">
2.2. Sampling and Analytical Procedures</div> <p>After 72 h of incubation, gas pressure above ambient was recorded, and a 12-mL gas sample was taken with a glass syringe and placed into vacutainers previously perfused with dinitrogen (N <sub>2</sub>) and evacuated to 0.2 atm. Bottles were then uncapped and final pH (Oakton <sup>®</sup> pH 700 meter, Vernon Hills, IL, USA) and reducing potential (E <i>h</i>) (Schott Instruments BlueLine 31 Rx Ag/AgCl redox electrode in saturated KCl) immediately measured, and 0.5 mL of a 20% ( <i>m</i>/ <i>v</i>) sodium azide (Merck 6688) solution was added to arrest microbial activity [ <a href="#B21-microorganisms-07-00115">21</a>]. One milliliter aliquots were transferred to eppendorfs containing 0.2 mL of 20% ( <i>m</i>/ <i>v</i>) m-phosphoric acid (Merck 1.00546.0500) or 1% ( <i>v</i>/ <i>v</i>) sulfuric acid, and kept at −20 °C until analyzed for VFA and ammonium ion (NH <sub>4</sub> <sup>+</sup>) concentration, respectively. Bottles were then shaken for a few seconds to re-suspend solid particles, and 1-mL aliquots were sampled using plastic Pasteur pipettes, and transferred to eppendorfs and stored at −80 °C until DNA was extracted for DGGE analysis. Bottles were again shaken for a few seconds to re-suspend solid particles and 10 mL were transferred to 15 mL Falcon tubes and kept frozen at −20 °C until lyophilized and analyzed for AA content. The remaining content of the bottles was centrifuged into pre-weighted bottles at 10,956× <i>g</i> and 6 °C for 20 min. The supernatant was discarded and the pellet was lyophilized. Tubes were then weighed, the pellet mass calculated by difference, and the pellet analyzed for DM, total ashes, total N, and NDF content [ <a href="#B22-microorganisms-07-00115">22</a>]. </p> <p>Gas samples were analyzed for their content of CH <sub>4</sub> and H <sub>2</sub> in a Clarus 580 PerkinElmer GC equipped with a 60/80 Carboxen 1000 (Supelco, Bellefonte, PA, USA) packed column and a thermal conductivity detector and an isothermal oven temperature of 180 °C, using N <sub>2</sub> at 30 mL/min as a carrier gas. Samples for VFA analysis were thawed, vortexed, and centrifuged at 16,000× <i>g</i> for 10 min. The supernatant was then filtered through 0.45 µm pore cellulose filters into 2 mL GC vials. One microliter VFA sample was injected in a PerkinElmer Clarus580 GC equipped with an Elite-FFAP (PerkinElmer, Shelton, CT, USA) capillary column and a flame ionization detector. Helium at 1.5 mL was the carrier gas. Initial temperature was 90 °C with a 12 °C/min ramp until 150 °C, which was held for 5 min. </p> <p>Ammonium concentration was determined colorimetrically according to Kaplan [ <a href="#B23-microorganisms-07-00115">23</a>]. Amino acids contents were analyzed in Falcon tubes containing 10-mL culture aliquots with suspended solid particles stored frozen. Tubes were lyophilized and subsamples of approximately 20 mg were transferred to hydrolysis tubes and added 1 mL of a 6 M hydrochloric acid and 1% ( <i>m</i>/ <i>v</i>) phenol solution. The tubes were then gassed with N <sub>2</sub>, closed air tightly, and incubated at 110 °C for 24 h [ <a href="#B24-microorganisms-07-00115">24</a>]. Tubes contents were filtered through 0.45 µm cellulose filters and 20 µL of the filtrate diluted with ultrapure water to 500 µL. Ten microliters of the resulting dilution were derivatized with 6-aminoquinolyl-N-hydroxysuccinimidylcarbamate using the Waters <sup>®</sup> AccQ·Tag <sup>TM</sup> Amino Acid Analysis Method (Waters Corporation, Milford, MA, USA). Subsequently, 20 µL of derivatized sample were injected into a Hitachi L-7100 HPLC equipped with a C18 Sunshell column and a UV-Visible detector operating at 254 nm. Oven temperature was 36 °C using sodium acetate and 60% ( <i>v</i>/ <i>v</i>) acetonitrile as mobile phases. A standard AA mix containing all AA except Gln, Asn, and Trp at 100 µM concentration (Waters Corporation, Milford, MA, USA) was used to fit standard curves for each AA. </p> </div> <div class="2.3.denaturinggradientgelelectrophoresisanalysis" title="sec"> <div class="title" tagx="title" title="title">
2.3. Denaturing Gradient Gel Electrophoresis Analysis</div> <p>Frozen samples for DGGE analysis from the third incubation run of the experiment were thawed and total microbial gDNA was extracted using the repeated bead-beating protocol of Yu and Forster [ <a href="#B25-microorganisms-07-00115">25</a>]. gDNA concentration was measured using a Maestrogen spectrophotometer (Maestrogen, Hsinchu City, Taiwan). The V3 region of the 16S rRNA bacterial gene was amplified by PCR using primers 341f-GC and 543r, following procedures described by Martínez et al. [ <a href="#B26-microorganisms-07-00115">26</a>]. Purified PCR products were quantified using the NanoQuant Plate (Tecan Group Ltd., Männedorf, Switzerland), following the manufacturer’s instructions. Samples of two replicates per treatment from incubation run three were run on two DGGE gels. To standardize the amount of PCR amplicon loaded onto the gel, 100 ng of the purified bacterial PCR product was loaded into each well of a DGGE gel. PCR products were separated on an 8% polyacrylamide gel with a denaturing gradient of 30 to 60% urea/formamide in a solution of 0.5× TAE buffer at 60 °C using the DCode System (Bio-Rad, Hercules, CA, USA), and run at 100 V for 18 h. A ladder with multiples bands was run in two separate positions in the gel to aid in normalization during gel analysis. The gel was stained using the silver staining method [ <a href="#B27-microorganisms-07-00115">27</a>]. Digitalized bacterial DGGE profiles were analyzed to calculate similarity matrices, and dendrograms with UPGMA algorithm for each gel based on band patterns with an optimization of 2% were constructed using Phoretix 1D (TotalLab Ltd., Newcastle upon Tyne, UK). </p> </div> <div class="2.4.calculations" title="sec"> <div class="title" tagx="title" title="title">
2.4. Calculations</div> <p>Total gas pressure in the serum bottles at the beginning and at the end of the incubations was calculated as the measured gas pressure plus 101,325 Pa (1 atm). The number of moles of total gas present at the beginning and at the end of the incubations was calculated using the ideal gas law [ <a href="#B28-microorganisms-07-00115">28</a>] considering a 120 mL headspace and the total gas pressure calculated as described above. Total gas production in moles was calculated by subtracting the initial from the final number of moles of total gas. The number of moles of CH <sub>4</sub> and H <sub>2</sub> present in the headspace were calculated by multiplying the number of moles of total gas at the end of the incubations by the analyzed volume percentage of each gas. The molar percentages of CH <sub>4</sub> and H <sub>2</sub> in total gas were then corrected and reported with respect to total gas actually produced. Because H <sub>2</sub> is an intermediate in rumen fermentation with a large turnover that is incorporated into various fermentation pathways [ <a href="#B3-microorganisms-07-00115">3</a>, <a href="#B29-microorganisms-07-00115">29</a>], H <sub>2</sub> is reported as H <sub>2</sub> accumulated, rather than as H <sub>2</sub> production. Recorded E <i>h</i> was corrected to the Standard Hydrogen Electrode (SHE) by adding 197 mv [ <a href="#B30-microorganisms-07-00115">30</a>]. </p> <p>Incubated cellulose contained 95% DM, and 0.03% total N, 97.7% NDF (DM basis), and undetectable total ashes. The amount of undigested cellulose substrate expressed as DM was calculated by dividing the NDF content in the undigested residue by cellulose NDF content. As incubated cellulose had undetectable total ash content, undigested cellulose DM was assumed to be equal to undigested cellulose organic matter (OM): </p><div class="disp-formula" title="disp-formula">
Undigested cellulose (mg DM) = undigested cellulose (mg OM) = undigested pellet (mg DM) × (NDF% in undigested pellet ÷ 100) ÷ 0.977 <span class="label" tagx="label" title="label">(1)</span> </div> <p /> <p>It was assumed that undigested cellulose had the same composition than the cellulose substrate. True digestibility of OM was calculated by dividing the difference between incubated OM in cellulose and undigested cellulose OM was calculated by Equation (1), expressing the result as a percentage. </p> <p> </p><div class="disp-formula" title="disp-formula">
OM true digestibility = [cellulose (mg OM) − undigested cellulose (mg OM)] × 100 ÷ cellulose (mg OM) <span class="label" tagx="label" title="label">(2)</span> </div> Microbial OM production was calculated by subtracting the mass of undigested cellulose OM from the total mass of the undigested solid residue OM: <div class="disp-formula" title="disp-formula">
Microbial biomass (mg OM) = undigested solid residue (mg OM) − undigested cellulose (mg OM) <span class="label" tagx="label" title="label">(3)</span> </div> <p /> <p>Net production of microbial N and AA were calculated by assuming that all N and AA present in the undigested residue corresponded to net microbial synthesis during the incubations, as their initial content was minimized by using pure cellulose as substrate and reducing the amount of inoculum from 20% ( <i>v</i>/ <i>v</i>) [ <a href="#B12-microorganisms-07-00115">12</a>] to 0.1%. </p> <p>Changes in [H] flows and sinks were quantified as pairs of reducing equivalents ([2H]) produced and incorporated. A balance of [2H] produced and incorporated in the catabolism of glucose (as the product of cellulose hydrolysis) to VFA and gases was calculated as by Ungerfeld [ <a href="#B9-microorganisms-07-00115">9</a>] and Guyader et al. [ <a href="#B21-microorganisms-07-00115">21</a>] ( <a href="#app1-microorganisms-07-00115">Supplementary Table S1</a>), except that formate and heptanoate were not considered because they were not determined, and caproate was not considered for simplification, as there are two different pathways of formation with different implications to the electron balance [ <a href="#B21-microorganisms-07-00115">21</a>]. </p> <p>Production and incorporation of [2H] was also calculated for the synthesis of AA from glucose, CO <sub>2</sub> (from bicarbonate in the medium), VFA added to the medium, ATP carbon (in case of His), and NH <sub>4</sub> <sup>+</sup> ( <a href="#app1-microorganisms-07-00115">Supplementary Table S2</a>). Calculation of production and incorporation of [2H] for each AA was based on synthetic pathways previously described [ <a href="#B1-microorganisms-07-00115">1</a>, <a href="#B11-microorganisms-07-00115">11</a>, <a href="#B31-microorganisms-07-00115">31</a>, <a href="#B32-microorganisms-07-00115">32</a>, <a href="#B33-microorganisms-07-00115">33</a>]. In some cases, the existence of more than one biosynthetic pathways for some AA can result in varying implications with regards to the uptake of disposal of [2H] [ <a href="#B1-microorganisms-07-00115">1</a>, <a href="#B29-microorganisms-07-00115">29</a>]. For some AA, various biosynthetic pathways were therefore considered ( <a href="#app1-microorganisms-07-00115">Supplementary Table S2</a>). In other cases, only the main pathway based on previous results with rumen microorganisms was considered. Transaminations were considered to indirectly incorporate 1 mol of [2H] per mol of NH <sub>4</sub> <sup>+</sup> because amination in the synthesis of the AA donating NH <sub>4</sub> <sup>+</sup> in the transamination reaction, generally glutamate or alanine, would incorporate 1 mol of [2H] [ <a href="#B1-microorganisms-07-00115">1</a>]. Maximal and minimal net [2H] incorporation into AA synthesis was calculated for each experimental unit. Finally, overall maximal and minimal recovery of [2H] was calculated considering VFA and gases and AA synthetic pathways that maximized and minimized [2H] incorporation, respectively. </p> </div> <div class="2.5.statisticalanalyses" title="sec"> <div class="title" tagx="title" title="title">
2.5. Statistical Analyses</div> <p>There were two replicates per treatment per incubation and four incubation runs conducted on different days. All responses were modelled as a function of the fixed effect of the treatment (methanogenesis inhibitor) and the random effect of the incubation run. If the treatment effect was significant ( <i>p</i> &amp;lt; 0.05), treatment means were separated using Tukey’s HSD. </p> <p>Also, [2H] recovery in (i) VFA + gases; (ii) VFA + gases + AA calculated with maximal net incorporation of [2H] into AA, and (iii) VFA + gases + AA calculated with maximal net incorporation of [2H] into AA, were regressed against CH <sub>4</sub> production per gram of OM incubated and truly digested. </p> <p>Homogeneity of variances was evaluated by examining plots of residuals against predicted. We used residual normality plots to examine the assumption of normality of residuals. Outliers were identified as those treatment means whose absolute value studentized residuals were greater than <i>t</i> <sub>68, 0.95</sub> (resulting from <i>t</i> <sub>N-P-1, 0.95</sub>, P being three parameters and N = 72 observations for a 95% confidence interval) and deleted from the analysis. </p> <p>JMP <sup>®</sup> 13.2.1 [ <a href="#B34-microorganisms-07-00115">34</a>] was used for all statistical analyses. </p> </div> </div> <div class="results" title="results"> <div class="title" tagx="title" title="title">
3. Results</div> <div class="3.1.fermentation" title="sec"> <div class="title" tagx="title" title="title">
3.1. Fermentation</div> <p>Inhibitors AQ, PA, and SN decreased ( <i>p</i> &amp;lt; 0.05) gas production and total VFA concentration and increased ( <i>p</i> &amp;lt; 0.05) final pH with respect to the control treatment ( <a href="#microorganisms-07-00115-t001">Table 1</a>). All additives except for linseed oil inhibited ( <i>p</i> &amp;lt; 0.05) CH <sub>4</sub> production, and BES and AQ caused accumulation of H <sub>2</sub> ( <i>p</i> &amp;lt; 0.05). AQ decreased final E <i>h</i> ( <i>p</i> &amp;lt; 0.05). Acetate molar percentage was increased ( <i>p</i> &amp;lt; 0.05) by PA and SN, and propionate decreased ( <i>p</i> &amp;lt; 0.05) by AQ, PA, E2B, and SN. Butyrate molar percentage was increased ( <i>p</i> &amp;lt; 0.05) by AQ and E2B and unaffected by the other additives. The additives had no effect ( <i>p</i> &amp;gt; 0.05) on isobutyrate or 2- and 3-methylbutyrate molar percentages or NH <sub>4</sub> <sup>+</sup> concentration. Valerate molar percentage was increased ( <i>p</i> &amp;lt; 0.05) by AQ, E2B and SN. Caproate molar percentage was increased ( <i>p</i> &amp;lt; 0.05) by PA. The acetate to propionate molar ratio was increased by PA and SN. </p> </div> <div class="3.2.digestionandmicrobialbiomassproductionandcomposition" title="sec"> <div class="title" tagx="title" title="title">
3.2. Digestion and Microbial Biomass Production and Composition</div> <p>True digestibility of OM was decreased ( <i>p</i> &amp;lt; 0.05) by E2B ( <a href="#microorganisms-07-00115-t002">Table 2</a>). There was no effect of the inhibitors on microbial OM production ( <i>p</i> = 0.14), and PA and SN decreased ( <i>p</i> &amp;lt; 0.001) microbial N production. </p> <p>There was a general effect of treatments on microbial AA N, but no pair of means was found different by Tukey HSD contrasts ( <a href="#microorganisms-07-00115-t002">Table 2</a>). The percentage of aspartate in total AA was increased ( <i>p</i> &amp;lt; 0.05) by SN. The percentages of serine, arginine, threonine, tyrosine, and leucine in total AA were decreased ( <i>p</i> &amp;lt; 0.05) by PA and SN. AQ and E2B increased ( <i>p</i> &amp;lt; 0.05) the percentage of alanine, and PA of proline, in total AA. E2B decreased ( <i>p</i> &amp;lt; 0.05) the percentage of phenylalanine in total AA. There were no effects of the inhibitors on the percentages of glutamate ( <i>p</i> = 0.067), isoleucine ( <i>p</i> = 0.058) or lysine ( <i>p</i> = 0.17) in total AA. </p> </div> <div class="3.3.reducingequivalentsbalance" title="sec"> <div class="title" tagx="title" title="title">
3.3. Reducing Equivalents Balance</div> <p>Production of [2H] in glucose fermentation to VFA and gases was decreased by PA and SN, whilst [2H] incorporation in VFA and gases and [2H] recovery were decreased by AQ, PA, E2B, and SN ( <a href="#microorganisms-07-00115-t003">Table 3</a>; <i>p</i> &amp;lt; 0.05). Methanogenesis inhibition caused a pronounced decrease ( <i>p</i> &amp;lt; 0.001) in [2H] recovery in VFA and gases ( <a href="#microorganisms-07-00115-f001">Figure 1</a> and <a href="#microorganisms-07-00115-f002">Figure 2</a>). There were no treatments effects on maximal ( <i>p</i> = 0.34) or minimal ( <i>p</i> = 0.16) net incorporation of [2H] into the synthesis of microbial AA. Inhibiting methanogenesis also decreased the overall recovery of [2H] considering the synthesis of microbial AA, whether net incorporation of [2H] into microbial AA was calculated to be maximal or minimal ( <a href="#microorganisms-07-00115-t003">Table 3</a> and <a href="#microorganisms-07-00115-f001">Figure 1</a> and <a href="#microorganisms-07-00115-f002">Figure 2</a>). Even though considering the incorporation of [2H] into the synthesis of microbial AA increased [2H] recovery, the response of [2H] recovery to methanogenesis inhibition was not altered ( <a href="#microorganisms-07-00115-f001">Figure 1</a> and <a href="#microorganisms-07-00115-f002">Figure 2</a>). </p> </div> <div class="3.4.denaturinggradientgelelectrophoresisanalysis" title="sec"> <div class="title" tagx="title" title="title">
3.4. Denaturing Gradient Gel Electrophoresis Analysis</div> <p>There was variation among treatments in the DGGE patterns of bands ( <a href="#microorganisms-07-00115-f003">Figure 3</a>). Bromotrichloromethane and chloroform clustered together. Bromoethanesulfonate showed a similar pattern of bands than the Control treatment. Sodium nitrate and AQ showed less intense band patterns than the Control and the other additives, with AQ forming a cluster by itself. A cluster of PA, E2B, and LO was also evidenced, with PA and E2B clustering closer. </p> </div> </div> <div class="discussion" title="discussion"> <div class="title" tagx="title" title="title">
4. Discussion</div> <p>We used a chemically pure, N-depleted substrate such as pure cellulose to be able to quantify the production of microbial N and AA without a microbial marker. We decreased both microbial and non-microbial N initially present in the incubations by drastically decreasing the proportion of inoculum from 20 [ <a href="#B12-microorganisms-07-00115">12</a>] to 0.1% ( <i>v</i>/ <i>v</i>). The rationale was that decreasing non-ammonium N initially present in the incubations would increase the proportion of N and AA in the incubation residue corresponding to microbial N, allowing to better evaluate the effects of the methanogenesis-inhibiting treatments. However, because such a pronounced decrease in the amount of inoculum would increase the lag time of the incubations substantially, the total incubation time was extended to 72 h based on preliminary results [ <a href="#B35-microorganisms-07-00115">35</a>]. </p> <p>Fermentation was atypical in some aspects, such as propionate being a more important sink of [H] than CH <sub>4</sub> in the Control treatment. The proportion of H <sub>2</sub> in total gas in the Control treatment was also high considering that it was only surpassed by BES and AQ. Also, [H] recovery both in the control treatment and under methanogenesis inhibition was lower than what was reported in a previous meta-analysis of methanogenesis inhibited batch cultures [ <a href="#B9-microorganisms-07-00115">9</a>]. The lower than typical importance of CH <sub>4</sub> as a [H] sink compared with propionate and H <sub>2</sub>, and the relatively small acetate to propionate molar ratio for a cellulose substrate, suggest that some conditions in the Control treatment might have negatively affected the growth of methanogens [ <a href="#B36-microorganisms-07-00115">36</a>]. It is possible that the small volume of inoculum, and perhaps the artificial nature of the pure cellulose substrate, resulted in deficiencies of vitamins or micronutrients which could have negatively affected methanogenesis, even though yeast extract and a complex micromineral solution were added to the medium intending to cover for possible deficiencies in minerals and vitamins. </p> <p>All additives except for linseed oil inhibited CH <sub>4</sub> production. Even though most additives also inhibited digestion and fermentation, as reflected by lower or numerically lower OM digestibility, total gas production, and total VFA concentration, CH <sub>4</sub> production was not decreased solely on the basis of less OM being digested, as CH <sub>4</sub> production per unit of truly digested OM was also decreased. A negative effect of linseed oil on CH <sub>4</sub> production had been expected because the predominant fatty acid in linseed oil is the highly unsaturated α-linolenic acid [ <a href="#B37-microorganisms-07-00115">37</a>], a strong inhibitor of methanogenesis in batch culture [ <a href="#B15-microorganisms-07-00115">15</a>, <a href="#B38-microorganisms-07-00115">38</a>]. It is possible that the small amount of inoculum used in this experiment was insufficient for hydrolyzing triacylglycerides in linseed oil to release substantial amounts of α-linolenic acid during the incubations in order to exert its toxicity to methanogens [ <a href="#B38-microorganisms-07-00115">38</a>]. </p> <p>Guyader et al. [ <a href="#B20-microorganisms-07-00115">20</a>] reported a linear increase in acetate molar percentage in rumen batch cultures with increasing initial concentration of nitrate, although propionate molar percentage and the acetate to propionate molar ratio were not affected in their study. Nitrate has a dual mechanism to inhibit methanogenesis, by competing for [H] as a strong electron acceptor in its reduction to NH <sub>4</sub> <sup>+</sup>, and through its reduction intermediate nitrite, which is toxic to methanogens [ <a href="#B39-microorganisms-07-00115">39</a>]. The observed increase in the acetate to propionate ratio and the numerical decrease in H <sub>2</sub> concentration suggests that the reduction of nitrate to nitrite and NH <sub>4</sub> <sup>+</sup> competed not only with CH <sub>4</sub> but also with propionate as a [H] sink. In the present experiment, total reduction of nitrate to NH <sub>4</sub> <sup>+</sup> would have potentially incorporated 5.2 mmol of [2H] and decreased CH <sub>4</sub> production by 1.3 mmol (calculations not shown), which is greater than the observed decrease in CH <sub>4</sub> production. However, complete reduction of nitrate to NH <sub>4</sub> <sup>+</sup> did not seem to occur, as the final NH <sub>4</sub> <sup>+</sup> concentration was not altered by SN. The unaffected final NH <sub>4</sub> <sup>+</sup> concentration was not a consequence of increased NH <sub>4</sub> <sup>+</sup> incorporation into microbial biomass, as SN resulted in lower microbial N production. Although nitrite concentration was not measured, it seems likely that much of added nitrate probably accumulated as nitrite, which could in turn explain the strong inhibition of fermentation reflected by the decrease in total VFA concentration in the SN treatment. Possibly, low availability of H <sub>2</sub> in the SN treatment limited the reduction of nitrate to NH <sub>4</sub> <sup>+</sup>. Part of N in added nitrate might have also been incorporated into other compounds such as nitrous oxide [ <a href="#B40-microorganisms-07-00115">40</a>]. </p> <p>Inhibiting methanogenesis is expected to cause a shift from acetate towards propionate production [ <a href="#B36-microorganisms-07-00115">36</a>], which has been confirmed in a meta-analysis of methanogenesis-inhibited batch cultures [ <a href="#B9-microorganisms-07-00115">9</a>]. Apart from SN, which, as discussed, acts as an alternative [H] sink to CH <sub>4</sub>, AQ, PA and E2B decreased CH <sub>4</sub> but instead of increasing propionate molar percentage they decreased it. In previous work, PA and E2B at initial concentrations similar to the present experiment increased propionate production [ <a href="#B19-microorganisms-07-00115">19</a>, <a href="#B41-microorganisms-07-00115">41</a>] although greater initial concentrations decreased propionate production [ <a href="#B41-microorganisms-07-00115">41</a>, <a href="#B42-microorganisms-07-00115">42</a>]. García-López et al. [ <a href="#B17-microorganisms-07-00115">17</a>] found increases or lack of effects of AQ at an initial concentration of up to 5 ppm in rumen batch cultures on propionate molar percentage, although 10 ppm in continuous cultures decreased propionate molar percentage. It seems that the greatest concentrations of AQ, PA, and E2B evaluated in the different studies may negatively affect succinate or propionate producers or both. In the present study, this might have been exacerbated by the small volume of inoculum, which could have decreased the initial microbial diversity in the incubations. The smallest final E <i>h</i> and greatest H <sub>2</sub> concentration in total gas caused by AQ suggests that the system had problems to dispose [H] into pathways alternative to methanogenesis such as propionate formation. </p> <p>The additives effects on digestion, fermentation, and microbial AA composition seemed to associate well with changes in the microbial community composition as reflected by the pattern of bands in the DGGE analysis. Chloroform and BTCM, both being chemical halogenated analogues of CH <sub>4</sub>, had similar effects on digestion, fermentation, and microbial AA composition, and clustered together in the DGGE analysis. Sodium nitrate and PA had similar effects on digestion and fermentation, inhibited the synthesis of the same AA, and also clustered together in the DGGE analysis. In previous work, however, PA and SN had clustered apart in a DGGE analysis [ <a href="#B42-microorganisms-07-00115">42</a>]. The exception to the associations between the effects on digestion, fermentation, and the AA profile with the DGGE pattern of bands was linseed oil, which was largely inert to digestion and fermentation but clustered apart from the Control treatment. </p> <p>Inhibiting rumen methanogenesis in vitro has been found to steadily decrease the recovery of [2H] in the sum of main [H] sinks CH <sub>4</sub>, propionate, butyrate, and the intermediate H <sub>2</sub>, which typically accumulates when CH <sub>4</sub> production is inhibited [ <a href="#B9-microorganisms-07-00115">9</a>]. Ungerfeld [ <a href="#B9-microorganisms-07-00115">9</a>] speculated that greater availability of [H] resulting from methanogenesis inhibition may stimulate the incorporation of NH <sub>4</sub> <sup>+</sup> into carbon chains for AA synthesis. Under the concentrations of NH <sub>4</sub> <sup>+</sup> generally found in the rumen, incorporation of NH <sub>4</sub> <sup>+</sup> into carbon chains predominantly involves low-affinity systems such as NADP- and NAD-glutamate dehydrogenases, and alanine dehydrogenases [ <a href="#B1-microorganisms-07-00115">1</a>], reactions that are coupled to the incorporation of [H] [ <a href="#B11-microorganisms-07-00115">11</a>]. Contrary to our hypothesis, however, inhibiting methanogenesis did not increase the production of microbial OM or N, nor the incorporation of [2H] into the synthesis of microbial AA. Some inhibitors altered the proportion of particular AA in the total, but a general consequence of methanogenesis inhibition on the AA profile could not be established. </p> <p>Taking into account microbial AA synthesis could increase [2H] recovery by between 1.0 and 5.7 percentage units (calculated as the differences in the intercepts in [2H] recovery in VFA and gases, and [2H] recovery in VFA and gases plus AA; <a href="#microorganisms-07-00115-f001">Figure 1</a> and <a href="#microorganisms-07-00115-f002">Figure 2</a>). It is important to consider that in the microbial cultures of the present experiment, the incorporation of [H] into AA biosynthesis was maximized by not providing preformed AA. However, the increase in [2H] recovery due to considering the incorporation of [2H] into AA synthesis was independent of the extent of methanogenesis inhibition, as reflected by the similar slopes of the responses of [2H] recovery in VFA and gases, and in VFA and gases plus AA, to CH <sub>4</sub> production ( <a href="#microorganisms-07-00115-f001">Figure 1</a> and <a href="#microorganisms-07-00115-f002">Figure 2</a>). Hence, at least under the conditions of this experiment, microbial AA biosynthesis did not contribute to explain the decrease in [2H] recovery caused by the inhibition of CH <sub>4</sub> production. Future experiments should consider studying changes in other fermentation products and processes that can incorporate [H], such as formate, lactate, succinate, alcohols, and reductive acetogenesis, and reduced cell components such as long chain fatty acids [ <a href="#B9-microorganisms-07-00115">9</a>]. </p> </div> <div class="conclusions" title="conclusions"> <div class="title" tagx="title" title="title">
5. Conclusions</div> <p>Under the conditions of this experiment, biosynthesis of microbial AA accounted for a minor, but not negligible percentage of [2H] recovery, but it did not explain the decline in the recovery of [2H] in main fermentation products that occurs when methanogenesis is inhibited. In general, changes in fermentation and microbial biomass production and AA profile caused by eight different inhibitors of methanogenesis associated with the changes in the microbial community composition that they seemed to cause. </p> </div> </div> <div class="back" title="back"> <div class="ack" title="ack"> <div class="title" tagx="title" title="title">
Acknowledgments</div> <p>The authors wish to thank Patricia Oliveira and Felipe Fuentes, Fernanda Soto, Felipe Fernández, Esteban Torres, Isaac Rabi, Raúl Dupretzz, Ignacia Miranda and Karen Peñailillo for technical assistance. </p> </div> <div class="app-group" title="app-group"> <div class="app" title="app"> <div class="title" tagx="title" title="title">
Supplementary Materials</div> <p>The following are available online at <span class="uri" title="uri"> <a href="https://www.mdpi.com/2076-2607/7/5/115/s1">https://www.mdpi.com/2076-2607/7/5/115/s1</a> </span>, Table S1: Pairs of reducing equivalents ([2H]) produced or incorporated associated to the formation of volatile fatty acids and gases (VFA) from glucose in rumen fermentation, Table S2: Pairs of reducing equivalents ([2H]) produced and incorporated associated to the formation of amino acids (AA) from glucose, carbon dioxide (CO <sub>2</sub>), preformed volatile fatty acids (VFA) and ammonium (NH <sub>4</sub> <sup>+</sup>) in rumen fermentation. </p> <div class="local-data" title="local-data"> <div class="media" title="media"> <a href="microorganisms-07-00115-s001.pdf">LINK</a>  <p>Click here for additional data file.</p>  </div> </div> </div> </div> <div class="title" tagx="title" title="title">
Author Contributions</div> <p>Conceptualization, E.M.U.; Methodology, E.M.U., M.F.A., E.D.M. and M.S.; Formal analysis, E.M.U., E.D.M. and M.S.; Investigation, E.M.U., M.F.A., E.D.M. and M.S.; Resources, E.M.U. and E.D.M.; Data curation, E.M.U., M.F.A., E.D.M. and M.S.; Writing—original draft preparation, E.M.U.; Writing—review and editing, E.M.U., M.F.A., E.D.M. and M.S.; Visualization, E.M.U., E.D.M. and M.S.; Supervision, E.M.U. and E.D.M.; Project administration, E.M.U.; Funding acquisition, E.M.U. </p> <div class="title" tagx="title" title="title">
Funding</div> <p>This research was funded by Comisión Nacional de Investigación Científica y Tecnológica (CONICYT), Santiago, Chile, Fondecyt 1160764 and the APC was funded by Comisión Nacional de Investigación Científica y Tecnológica (CONICYT), Santiago, Chile, Project Fondecyt 1160764. </p> <div class="title" tagx="title" title="title">
Conflicts of Interest</div> <p>The authors declare no conflict of interest. The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript, or in the decision to publish the results. </p> <div class="references">
References</div> <div tag="ref-list"> <ul> <div class="title" tagx="title" title="title">
References</div> <li tag="ref"> <a name="B1-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">1.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wallace</span> <span class="given-names" tagx="given-names" title="given-names">R.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Onodera</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cotta</span> <span class="given-names" tagx="given-names" title="given-names">M.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Metabolism of nitrogen-containing compounds</span> <span class="source" tagx="source" title="source">The Rumen Microbial Ecosystem</span> <span class="edition" tagx="edition" title="edition">2nd ed.</span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hobson</span> <span class="given-names" tagx="given-names" title="given-names">P.N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stewart</span> <span class="given-names" tagx="given-names" title="given-names">C.S.</span> </span> </span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Blackie Academic &amp;amp; Professional</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London, UK</span> <span class="year" tagx="year" title="year">1997</span> <span class="fpage" tagx="fpage" title="fpage">283</span> <span class="lpage" tagx="lpage" title="lpage">328</span> </span> </li> <li tag="ref"> <a name="B2-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">2.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wolin</span> <span class="given-names" tagx="given-names" title="given-names">M.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Miller</span> <span class="given-names" tagx="given-names" title="given-names">T.L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stewart</span> <span class="given-names" tagx="given-names" title="given-names">C.S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Microbe-microbe interactions</span> <span class="source" tagx="source" title="source">The Rumen Microbial Ecosystem</span> <span class="edition" tagx="edition" title="edition">2nd ed.</span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hobson</span> <span class="given-names" tagx="given-names" title="given-names">P.N.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Stewart</span> <span class="given-names" tagx="given-names" title="given-names">C.S.</span> </span> </span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Blackie Academic &amp;amp; Professional</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">London, UK</span> <span class="year" tagx="year" title="year">1997</span> <span class="fpage" tagx="fpage" title="fpage">467</span> <span class="lpage" tagx="lpage" title="lpage">491</span> </span> </li> <li tag="ref"> <a name="B3-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">3.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Greening</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Geier</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Woods</span> <span class="given-names" tagx="given-names" title="given-names">L.C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morales</span> <span class="given-names" tagx="given-names" title="given-names">S.E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McDonald</span> <span class="given-names" tagx="given-names" title="given-names">M.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rushton-Green</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morgan</span> <span class="given-names" tagx="given-names" title="given-names">X.C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Koike</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Leahy</span> <span class="given-names" tagx="given-names" title="given-names">S.C.</span> </span> <span class="etal" title="etal"> <i>et al.</i> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Alternative hydrogen uptake pathways suppress methane production in ruminants</span> <span class="source" tagx="source" title="source">bioRxiv</span> <span class="year" tagx="year" title="year">2018</span> <span class="fpage" tagx="fpage" title="fpage">486894</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1101/486894">10.1101/486894</a> </span> </span> </li> <li tag="ref"> <a name="B4-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">4.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moss</span> <span class="given-names" tagx="given-names" title="given-names">A.R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Jouany</span> <span class="given-names" tagx="given-names" title="given-names">J.-P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Newbold</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Methane production by ruminants: Its contribution to global warming</span> <span class="source" tagx="source" title="source">Ann. Zootech.</span> <span class="year" tagx="year" title="year">2000</span> <span class="volume" tagx="volume" title="volume">49</span> <span class="fpage" tagx="fpage" title="fpage">231</span> <span class="lpage" tagx="lpage" title="lpage">253</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1051/animres:2000119">10.1051/animres:2000119</a> </span> </span> </li> <li tag="ref"> <a name="B5-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">5.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">K.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Johnson</span> <span class="given-names" tagx="given-names" title="given-names">D.E.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Methane emissions from cattle</span> <span class="source" tagx="source" title="source">J. Anim. Sci.</span> <span class="year" tagx="year" title="year">1995</span> <span class="volume" tagx="volume" title="volume">73</span> <span class="fpage" tagx="fpage" title="fpage">2483</span> <span class="lpage" tagx="lpage" title="lpage">2492</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2527/1995.7382483x">10.2527/1995.7382483x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8567486">8567486</a> </span> </span> </li> <li tag="ref"> <a name="B6-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">6.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eckard</span> <span class="given-names" tagx="given-names" title="given-names">R.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Grainger</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">de Klein</span> <span class="given-names" tagx="given-names" title="given-names">C.A.M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Options for the abatement of methane and nitrous oxide from ruminant production: A review </span> <span class="source" tagx="source" title="source">Livestock Sci.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">130</span> <span class="fpage" tagx="fpage" title="fpage">47</span> <span class="lpage" tagx="lpage" title="lpage">56</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.livsci.2010.02.010">10.1016/j.livsci.2010.02.010</a> </span> </span> </li> <li tag="ref"> <a name="B7-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">7.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guyader</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Eugene</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meunier</span> <span class="given-names" tagx="given-names" title="given-names">B.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Doreau</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Morgavi</span> <span class="given-names" tagx="given-names" title="given-names">D.P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Silberberg</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rochette</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Gerard</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Loncke</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Additive methane-mitigating effect between linseed oil and nitrate fed to cattle</span> <span class="source" tagx="source" title="source">J. Anim. Sci.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">93</span> <span class="fpage" tagx="fpage" title="fpage">3564</span> <span class="lpage" tagx="lpage" title="lpage">3577</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2527/jas.2014-8196">10.2527/jas.2014-8196</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26440025">26440025</a> </span> </span> </li> <li tag="ref"> <a name="B8-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">8.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ungerfeld</span> <span class="given-names" tagx="given-names" title="given-names">E.M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Inhibition of rumen methanogenesis and ruminant productivity: A meta-analysis</span> <span class="source" tagx="source" title="source">Front. Vet. Sci.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">5</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">13</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fvets.2018.00113">10.3389/fvets.2018.00113</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29417054">29417054</a> </span> </span> </li> <li tag="ref"> <a name="B9-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">9.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ungerfeld</span> <span class="given-names" tagx="given-names" title="given-names">E.M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Shifts in metabolic hydrogen sinks in the methanogenesis-inhibited ruminal fermentation: A meta-analysis </span> <span class="source" tagx="source" title="source">Front. Microbiol.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">6</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">17</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fmicb.2015.00037">10.3389/fmicb.2015.00037</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/25653648">25653648</a> </span> </span> </li> <li tag="ref"> <a name="B10-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">10.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hino</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Russell</span> <span class="given-names" tagx="given-names" title="given-names">J.B.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effect of reducing-equivalent disposal and NADH/NAD on deamination of amino acids by intact rumen microorganisms and their cell extracts </span> <span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span> <span class="year" tagx="year" title="year">1985</span> <span class="volume" tagx="volume" title="volume">50</span> <span class="fpage" tagx="fpage" title="fpage">1368</span> <span class="lpage" tagx="lpage" title="lpage">1374</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/4091565">4091565</a> </span> </span> </li> <li tag="ref"> <a name="B11-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">11.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voet</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Voet</span> <span class="given-names" tagx="given-names" title="given-names">J.G.</span> </span> </span> <span class="source" tagx="source" title="source">Biochemistry</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">John Wiley &amp;amp; Sons, Inc.</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">New York, NY, USA</span> <span class="year" tagx="year" title="year">1995</span> <span class="fpage" tagx="fpage" title="fpage">1392</span> </span> </li> <li tag="ref"> <a name="B12-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">12.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Goering</span> <span class="given-names" tagx="given-names" title="given-names">H.K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">van Soest</span> <span class="given-names" tagx="given-names" title="given-names">P.J.</span> </span> </span> <span class="source" tagx="source" title="source">Forage Fiber Analyses (Apparatus, Reagents, Procedures, and Some Applications)</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">United States Department of Agriculture</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Washington, DC, USA</span> <span class="year" tagx="year" title="year">1975</span> <span class="fpage" tagx="fpage" title="fpage">24</span> </span> </li> <li tag="ref"> <a name="B13-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">13.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Schaefer</span> <span class="given-names" tagx="given-names" title="given-names">D.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Davis</span> <span class="given-names" tagx="given-names" title="given-names">C.L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bryant</span> <span class="given-names" tagx="given-names" title="given-names">M.P.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Ammonia saturation constants for predominant species of rumen bacteria</span> <span class="source" tagx="source" title="source">J. Dairy Sci.</span> <span class="year" tagx="year" title="year">1980</span> <span class="volume" tagx="volume" title="volume">63</span> <span class="fpage" tagx="fpage" title="fpage">1248</span> <span class="lpage" tagx="lpage" title="lpage">1263</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3168/jds.S0022-0302(80)83076-1">10.3168/jds.S0022-0302(80)83076-1</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/7419777">7419777</a> </span> </span> </li> <li tag="ref"> <a name="B14-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">14.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Caldwell</span> <span class="given-names" tagx="given-names" title="given-names">D.R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bryant</span> <span class="given-names" tagx="given-names" title="given-names">M.P.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Medium without rumen fluid for nonselective enumeration and isolation of rumen bacteria</span> <span class="source" tagx="source" title="source">Appl. Microbiol.</span> <span class="year" tagx="year" title="year">1966</span> <span class="volume" tagx="volume" title="volume">14</span> <span class="fpage" tagx="fpage" title="fpage">794</span> <span class="lpage" tagx="lpage" title="lpage">801</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/5970467">5970467</a> </span> </span> </li> <li tag="ref"> <a name="B15-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">15.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O’Brien</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Navarro-Villa</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Purcell</span> <span class="given-names" tagx="given-names" title="given-names">P.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Boland</span> <span class="given-names" tagx="given-names" title="given-names">T.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">O’Kiely</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Reducing <i>in vitro</i> rumen methanogenesis for two contrasting diets using a series of inclusion rates of different additives </span> <span class="source" tagx="source" title="source">Anim. Prod. Sci.</span> <span class="year" tagx="year" title="year">2013</span> <span class="volume" tagx="volume" title="volume">54</span> <span class="fpage" tagx="fpage" title="fpage">141</span> <span class="lpage" tagx="lpage" title="lpage">157</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1071/AN12204">10.1071/AN12204</a> </span> </span> </li> <li tag="ref"> <a name="B16-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">16.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sauer</span> <span class="given-names" tagx="given-names" title="given-names">F.D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Teather</span> <span class="given-names" tagx="given-names" title="given-names">R.M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Changes in oxidation reduction potentials and volatile fatty acids production by rumen bacteria when methane synthesis is inhibited </span> <span class="source" tagx="source" title="source">J. Dairy Sci.</span> <span class="year" tagx="year" title="year">1987</span> <span class="volume" tagx="volume" title="volume">70</span> <span class="fpage" tagx="fpage" title="fpage">1835</span> <span class="lpage" tagx="lpage" title="lpage">1840</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3168/jds.S0022-0302(87)80222-9">10.3168/jds.S0022-0302(87)80222-9</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/2822785">2822785</a> </span> </span> </li> <li tag="ref"> <a name="B17-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">17.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">García-López</span> <span class="given-names" tagx="given-names" title="given-names">P.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kung</span> <span class="given-names" tagx="given-names" title="given-names">L.</span> <span class="suffix" tagx="suffix" title="suffix">Jr.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Odom</span> <span class="given-names" tagx="given-names" title="given-names">J.M.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">In vitro inhibition of microbial methane production by 9,10-anthraquinone</span> <span class="source" tagx="source" title="source">J. Anim. Sci.</span> <span class="year" tagx="year" title="year">1996</span> <span class="volume" tagx="volume" title="volume">74</span> <span class="fpage" tagx="fpage" title="fpage">2276</span> <span class="lpage" tagx="lpage" title="lpage">2284</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2527/1996.7492276x">10.2527/1996.7492276x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8880433">8880433</a> </span> </span> </li> <li tag="ref"> <a name="B18-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">18.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Bauchop</span> <span class="given-names" tagx="given-names" title="given-names">T.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Inhibition of rumen methanogenesis by methane analogues</span> <span class="source" tagx="source" title="source">J. Bacteriol.</span> <span class="year" tagx="year" title="year">1967</span> <span class="volume" tagx="volume" title="volume">94</span> <span class="fpage" tagx="fpage" title="fpage">171</span> <span class="lpage" tagx="lpage" title="lpage">175</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/6067522">6067522</a> </span> </span> </li> <li tag="ref"> <a name="B19-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">19.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ungerfeld</span> <span class="given-names" tagx="given-names" title="given-names">E.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rust</span> <span class="given-names" tagx="given-names" title="given-names">S.R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burnett</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effects of butyrate precursors on electron relocation when methanogenesis is inhibited in ruminal mixed cultures </span> <span class="source" tagx="source" title="source">Let. Appl. Microbiol.</span> <span class="year" tagx="year" title="year">2006</span> <span class="volume" tagx="volume" title="volume">42</span> <span class="fpage" tagx="fpage" title="fpage">567</span> <span class="lpage" tagx="lpage" title="lpage">572</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1472-765X.2006.01890.x">10.1111/j.1472-765X.2006.01890.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/16706893">16706893</a> </span> </span> </li> <li tag="ref"> <a name="B20-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">20.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guyader</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Tavendale</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Muetzel</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Dose-response effect of nitrate on hydrogen distribution between rumen fermentation end products: an <i>in vitro</i> approach </span> <span class="source" tagx="source" title="source">Anim. Prod. Sci.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">56</span> <span class="fpage" tagx="fpage" title="fpage">224</span> <span class="lpage" tagx="lpage" title="lpage">230</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1071/AN15526">10.1071/AN15526</a> </span> </span> </li> <li tag="ref"> <a name="B21-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">21.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guyader</span> <span class="given-names" tagx="given-names" title="given-names">J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ungerfeld</span> <span class="given-names" tagx="given-names" title="given-names">E.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Beauchemin</span> <span class="given-names" tagx="given-names" title="given-names">K.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Redirection of metabolic hydrogen by inhibiting methanogenesis in the rumen simulation technique (RUSITEC) </span> <span class="source" tagx="source" title="source">Front. Microbiol.</span> <span class="year" tagx="year" title="year">2017</span> <span class="volume" tagx="volume" title="volume">8</span> <span class="fpage" tagx="fpage" title="fpage">393</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fmicb.2017.00393">10.3389/fmicb.2017.00393</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/28352256">28352256</a> </span> </span> </li> <li tag="ref"> <a name="B22-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">22.</span> <span class="element-citation'"> <span class="person-group'"> <span class="collab'">collab: AOAC</span> </span> <span class="source" tagx="source" title="source">Official Methods of Analysis</span> <span class="edition" tagx="edition" title="edition">16th ed.</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Association of Official Analytical Chemists (AOAC) International</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Airlington, VA, USA</span> <span class="year" tagx="year" title="year">1995</span> <span class="fpage" tagx="fpage" title="fpage">771</span> </span> </li> <li tag="ref"> <a name="B23-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">23.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kaplan</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">The determination of urea, ammonia, and urease</span> <span class="source" tagx="source" title="source">Methods Biochem. Anal.</span> <span class="year" tagx="year" title="year">1969</span> <span class="volume" tagx="volume" title="volume">17</span> <span class="fpage" tagx="fpage" title="fpage">311</span> <span class="lpage" tagx="lpage" title="lpage">324</span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/4893472">4893472</a> </span> </span> </li> <li tag="ref"> <a name="B24-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">24.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Molina-Alcaide</span> <span class="given-names" tagx="given-names" title="given-names">E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Moumen</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martin-Garcia</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Carro</span> <span class="given-names" tagx="given-names" title="given-names">M.D.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Comparison of bacterial pellets and microbial markers for the estimation of the microbial nitrogen and amino acids flows from single flow continuous culture fermenters fed diets containing two-stage olive cake </span> <span class="source" tagx="source" title="source">J. Anim. Physiol. Anim. Nutr. (Berl.)</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">93</span> <span class="fpage" tagx="fpage" title="fpage">527</span> <span class="lpage" tagx="lpage" title="lpage">537</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1111/j.1439-0396.2008.00834.x">10.1111/j.1439-0396.2008.00834.x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/18537852">18537852</a> </span> </span> </li> <li tag="ref"> <a name="B25-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">25.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Forster</span> <span class="given-names" tagx="given-names" title="given-names">R.J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nucleic acid extraction, oligonucleotide probes and PCR methods</span> <span class="source" tagx="source" title="source">Methods in Gut Microbial Ecology for Ruminants</span> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Makkar</span> <span class="given-names" tagx="given-names" title="given-names">H.P.S.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">McSweeney</span> <span class="given-names" tagx="given-names" title="given-names">C.S.</span> </span> </span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">International Atomic Energy Agency, FAO, Springer</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Dordrecht, The Netherlands</span> <span class="year" tagx="year" title="year">2005</span> <span class="fpage" tagx="fpage" title="fpage">81</span> <span class="lpage" tagx="lpage" title="lpage">104</span> </span> </li> <li tag="ref"> <a name="B26-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">26.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Martínez</span> <span class="given-names" tagx="given-names" title="given-names">E.D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Turnbull</span> <span class="given-names" tagx="given-names" title="given-names">K.E.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Quigley</span> <span class="given-names" tagx="given-names" title="given-names">S.P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Streeter</span> <span class="given-names" tagx="given-names" title="given-names">S.J.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Swain</span> <span class="given-names" tagx="given-names" title="given-names">A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Klieve</span> <span class="given-names" tagx="given-names" title="given-names">A.V.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ouwerkerk</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poppi</span> <span class="given-names" tagx="given-names" title="given-names">D.P.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Liquid phase DGGE profiles of rumen bacteria from Brahman cross steers selected into two groups on the basis of post-weaning live weight gain on low crude protein pasture </span> <span class="source" tagx="source" title="source">Anim. Prod. Sci.</span> <span class="year" tagx="year" title="year">2012</span> <span class="volume" tagx="volume" title="volume">52</span> <span class="fpage" tagx="fpage" title="fpage">647</span> <span class="lpage" tagx="lpage" title="lpage">652</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1071/AN11234">10.1071/AN11234</a> </span> </span> </li> <li tag="ref"> <a name="B27-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">27.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kocherginskaya</span> <span class="given-names" tagx="given-names" title="given-names">S.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Aminov</span> <span class="given-names" tagx="given-names" title="given-names">R.I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">White</span> <span class="given-names" tagx="given-names" title="given-names">B.A.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Analysis of the rumen bacterial diversity under two different diet conditions using denaturing gradient gel electrophoresis, random sequencing, and statistical ecology approaches </span> <span class="source" tagx="source" title="source">Anaerobe</span> <span class="year" tagx="year" title="year">2001</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">119</span> <span class="lpage" tagx="lpage" title="lpage">134</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1006/anae.2001.0378">10.1006/anae.2001.0378</a> </span> </span> </li> <li tag="ref"> <a name="B28-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">28.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Singh</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">White</span> <span class="given-names" tagx="given-names" title="given-names">D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Demirel</span> <span class="given-names" tagx="given-names" title="given-names">Y.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Kelly</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Noll</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Blum</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Uncoupling Fermentative Synthesis of Molecular Hydrogen from Biomass Formation in <i>Thermotoga maritima</i> </span> <span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span> <span class="year" tagx="year" title="year">2018</span> <span class="volume" tagx="volume" title="volume">84</span> <span class="fpage" tagx="fpage" title="fpage">e00998-18</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.00998-18">10.1128/AEM.00998-18</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/29959252">29959252</a> </span> </span> </li> <li tag="ref"> <a name="B29-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">29.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Czerkawski</span> <span class="given-names" tagx="given-names" title="given-names">J.W.</span> </span> </span> <span class="source" tagx="source" title="source">An Introduction to Rumen Studies</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">Pergamon Press</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Oxford, UK</span> <span class="year" tagx="year" title="year">1986</span> <span class="fpage" tagx="fpage" title="fpage">246</span> </span> </li> <li tag="ref"> <a name="B30-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">30.</span> <span class="element-citation'"> <span class="mixed-article-title" title="mixed-article-title">Silver Chloride Electrode</span> <span class="comment">Available online: <a href="https://en.wikipedia.org/wiki/Silver_chloride_electrode#cite_note-5">https://en.wikipedia.org/wiki/Silver_chloride_electrode#cite_note-5</a> </span> <span class="date-in-citation" tagx="date-in-citation" title="date-in-citation">(accessed on 24 March 2019)</span> </span> </li> <li tag="ref"> <a name="B31-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">31.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Sauer</span> <span class="given-names" tagx="given-names" title="given-names">F.D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Erfle</span> <span class="given-names" tagx="given-names" title="given-names">J.D.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Mahadevan</span> <span class="given-names" tagx="given-names" title="given-names">S.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Amino acid biosynthesis in mixed rumen cultures</span> <span class="source" tagx="source" title="source">Annu. Rev. Biochem.</span> <span class="year" tagx="year" title="year">1975</span> <span class="volume" tagx="volume" title="volume">150</span> <span class="fpage" tagx="fpage" title="fpage">357</span> <span class="lpage" tagx="lpage" title="lpage">372</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1042/bj1500357">10.1042/bj1500357</a> </span> </span> </li> <li tag="ref"> <a name="B32-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">32.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Umbarger</span> <span class="given-names" tagx="given-names" title="given-names">H.E.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Amino acid biosynthesis and its regulation</span> <span class="source" tagx="source" title="source">Annu. Rev. Biochem.</span> <span class="year" tagx="year" title="year">1978</span> <span class="volume" tagx="volume" title="volume">47</span> <span class="fpage" tagx="fpage" title="fpage">532</span> <span class="lpage" tagx="lpage" title="lpage">606</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1146/annurev.bi.47.070178.002533">10.1146/annurev.bi.47.070178.002533</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/354503">354503</a> </span> </span> </li> <li tag="ref"> <a name="B33-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">33.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Allison</span> <span class="given-names" tagx="given-names" title="given-names">M.J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Biosynthesis of amino acids by ruminal microorganisms</span> <span class="source" tagx="source" title="source">J. Anim. Sci.</span> <span class="year" tagx="year" title="year">1969</span> <span class="volume" tagx="volume" title="volume">29</span> <span class="fpage" tagx="fpage" title="fpage">797</span> <span class="lpage" tagx="lpage" title="lpage">807</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2527/jas1969.295797x">10.2527/jas1969.295797x</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/4312543">4312543</a> </span> </span> </li> <li tag="ref"> <a name="B34-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">34.</span> <span class="element-citation'"> <span class="source" tagx="source" title="source">JMP <sup>®</sup> </span> <span class="comment">version 13.2.1</span> <span class="publisher-name" tagx="publisher-name" title="publisher-name">SAS Institute</span> <span class="publisher-loc" tagx="publisher-loc" title="publisher-loc">Cary, NC, USA</span> <span class="year" tagx="year" title="year">2016</span> </span> </li> <li tag="ref"> <a name="B35-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">35.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ungerfeld</span> <span class="given-names" tagx="given-names" title="given-names">E.M.</span> </span> <span class="citation_author_institution" id="">[], </span> </span> <span class="mixed-article-title" title="mixed-article-title">Preliminary experiment</span> <span class="year" tagx="year" title="year">2016</span> <span class="comment">unpublished</span> </span> </li> <li tag="ref"> <a name="B36-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">36.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Janssen</span> <span class="given-names" tagx="given-names" title="given-names">P.H.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Influence of hydrogen on rumen methane formation and fermentation balances through microbial growth kinetics and fermentation thermodynamics </span> <span class="source" tagx="source" title="source">Anim. Feed Sci. Technol.</span> <span class="year" tagx="year" title="year">2010</span> <span class="volume" tagx="volume" title="volume">160</span> <span class="fpage" tagx="fpage" title="fpage">1</span> <span class="lpage" tagx="lpage" title="lpage">22</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.anifeedsci.2010.07.002">10.1016/j.anifeedsci.2010.07.002</a> </span> </span> </li> <li tag="ref"> <a name="B37-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">37.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Prasad</span> <span class="given-names" tagx="given-names" title="given-names">K.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Flaxseed and cardiovascular health</span> <span class="source" tagx="source" title="source">J. Cardiovasc. Pharmacol.</span> <span class="year" tagx="year" title="year">2009</span> <span class="volume" tagx="volume" title="volume">54</span> <span class="fpage" tagx="fpage" title="fpage">369</span> <span class="lpage" tagx="lpage" title="lpage">377</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1097/FJC.0b013e3181af04e5">10.1097/FJC.0b013e3181af04e5</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19568181">19568181</a> </span> </span> </li> <li tag="ref"> <a name="B38-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">38.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhang</span> <span class="given-names" tagx="given-names" title="given-names">C.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Guo</span> <span class="given-names" tagx="given-names" title="given-names">Y.Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yuan</span> <span class="given-names" tagx="given-names" title="given-names">Z.P.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wu</span> <span class="given-names" tagx="given-names" title="given-names">Y.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wang</span> <span class="given-names" tagx="given-names" title="given-names">J.K.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Liu</span> <span class="given-names" tagx="given-names" title="given-names">J.X.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">W.Y.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effect of octadeca carbon fatty acids on microbial fermentation, methanogenesis and microbial flora in vitro </span> <span class="source" tagx="source" title="source">Anim. Feed Sci. Technol.</span> <span class="year" tagx="year" title="year">2008</span> <span class="volume" tagx="volume" title="volume">146</span> <span class="fpage" tagx="fpage" title="fpage">259</span> <span class="lpage" tagx="lpage" title="lpage">269</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1016/j.anifeedsci.2008.01.005">10.1016/j.anifeedsci.2008.01.005</a> </span> </span> </li> <li tag="ref"> <a name="B39-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">39.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yang</span> <span class="given-names" tagx="given-names" title="given-names">C.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rooke</span> <span class="given-names" tagx="given-names" title="given-names">J.A.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Cabeza</span> <span class="given-names" tagx="given-names" title="given-names">I.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Wallace</span> <span class="given-names" tagx="given-names" title="given-names">R.J.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Nitrate and inhibition of ruminal methanogenesis: Microbial ecology, obstacles, and opportunities for lowering methane emissions from ruminant livestock </span> <span class="source" tagx="source" title="source">Front. Microbiol.</span> <span class="year" tagx="year" title="year">2016</span> <span class="volume" tagx="volume" title="volume">7</span> <span class="fpage" tagx="fpage" title="fpage">132</span> <span class="pub-id"> <a href="https://dx.doi.org/10.3389/fmicb.2016.00132">10.3389/fmicb.2016.00132</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26904008">26904008</a> </span> </span> </li> <li tag="ref"> <a name="B40-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">40.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Petersen</span> <span class="given-names" tagx="given-names" title="given-names">S.O.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hellwing</span> <span class="given-names" tagx="given-names" title="given-names">A.L.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Brask</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Hojberg</span> <span class="given-names" tagx="given-names" title="given-names">O.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Poulsen</span> <span class="given-names" tagx="given-names" title="given-names">M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhu</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Baral</span> <span class="given-names" tagx="given-names" title="given-names">K.R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Lund</span> <span class="given-names" tagx="given-names" title="given-names">P.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Dietary nitrate for methane mitigation leads to nitrous oxide emissions from dairy cows </span> <span class="source" tagx="source" title="source">J. Environ. Qual.</span> <span class="year" tagx="year" title="year">2015</span> <span class="volume" tagx="volume" title="volume">44</span> <span class="fpage" tagx="fpage" title="fpage">1063</span> <span class="lpage" tagx="lpage" title="lpage">1070</span> <span class="pub-id"> <a href="https://dx.doi.org/10.2134/jeq2015.02.0107">10.2134/jeq2015.02.0107</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/26437087">26437087</a> </span> </span> </li> <li tag="ref"> <a name="B41-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">41.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Ungerfeld</span> <span class="given-names" tagx="given-names" title="given-names">E.M.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Rust</span> <span class="given-names" tagx="given-names" title="given-names">S.R.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Burnett</span> <span class="given-names" tagx="given-names" title="given-names">R.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Use of some novel alternative electron sinks to inhibit ruminal methanogenesis</span> <span class="source" tagx="source" title="source">Reprod. Nutr. Dev.</span> <span class="year" tagx="year" title="year">2003</span> <span class="volume" tagx="volume" title="volume">43</span> <span class="fpage" tagx="fpage" title="fpage">189</span> <span class="lpage" tagx="lpage" title="lpage">202</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1051/rnd:2003016">10.1051/rnd:2003016</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/12956318">12956318</a> </span> </span> </li> <li tag="ref"> <a name="B42-microorganisms-07-00115" /> <span class="label" tagx="label" title="label">42.</span> <span class="element-citation'"> <span class="person-group'"> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Zhou</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Meng</span> <span class="given-names" tagx="given-names" title="given-names">Q.</span> </span> <span class="name" tagx="name" title="name"> <span class="surname" tagx="surname" title="surname">Yu</span> <span class="given-names" tagx="given-names" title="given-names">Z.</span> </span> </span> <span class="mixed-article-title" title="mixed-article-title">Effects of methanogenic inhibitors on methane production and abundances of methanogens and cellulolytic bacteria in <i>in vitro</i> ruminal cultures </span> <span class="source" tagx="source" title="source">Appl. Environ. Microbiol.</span> <span class="year" tagx="year" title="year">2011</span> <span class="volume" tagx="volume" title="volume">77</span> <span class="fpage" tagx="fpage" title="fpage">2634</span> <span class="lpage" tagx="lpage" title="lpage">2639</span> <span class="pub-id"> <a href="https://dx.doi.org/10.1128/AEM.02779-10">10.1128/AEM.02779-10</a> </span> <span class="pub-id"> <a href="http://www.ncbi.nlm.nih.gov/pubmed/21357427">21357427</a> </span> </span> </li> </ul> </div> </div> <div class="floats-group" title="floats-group"> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 1</span> </a> </div>   <p>Response of recovery of reducing equivalents ([2H]) (%) to methane (CH <sub>4</sub>) produced per gram of organic matter (OM) incubated. Solid lines represent responses and dotted lines are 95% confidence bands. [2H] recovery (%) into: (i) Volatile fatty acids (VFA) and gases: • <i>y</i> = 36.9 (±1.24; <i>p</i> &amp;lt; 0.001) + incubation (random, <i>p</i> = 0.46) + 0.10 (±0.010; <i>p</i> &amp;lt; 0.001) <i>x</i>; <i>R</i> <sup>2</sup> = 0.66; (ii) VFA, gases and amino acids (AA) with maximum incorporation of [2H] into AA synthesis: <span class="styled-content" tagx="styled-content" title="styled-content">•</span> <i>y</i> = 42.6 (±2.11; <i>p</i> &amp;lt; 0.001) + incubation (random, <i>p</i> = 0.41) + 0.089 (±0.0095; <i>p</i> &amp;lt; 0.001) <i>x</i>; <i>R</i> <sup>2</sup> = 0.69; (iii) VFA, gases and AA with minimum incorporation of [2H] into AA synthesis: <span class="styled-content" tagx="styled-content" title="styled-content">•</span> <i>y</i> = 37.9 (±1.23; <i>p</i> &amp;lt; 0.001) + incubation (random, <i>p</i> = 0.80) + 0.095 (±0.0095; <i>p</i> &amp;lt; 0.001) <i>x</i>; <i>R</i> <sup>2</sup> = 0.68. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 2</span> </a> </div>   <p>Response of recovery of reducing equivalents ([2H]) (%) to CH <sub>4</sub> produced per gram of OM truly digested. Solid lines represent responses and dotted lines are 95% confidence bands. [2H] recovery (%) into: (i) VFA and gases: • <i>y</i> = 37.0 (±1.82; <i>p</i> &amp;lt; 0.001) + incubation (random, <i>p</i> = 0.77) + 0.041 (±0.0050; <i>p</i> &amp;lt; 0.001) <i>x</i>; <i>R</i> <sup>2</sup> = 0.58; (ii) VFA, gases and amino acids (AA) with maximum incorporation of [2H] into AA synthesis: <span class="styled-content" tagx="styled-content" title="styled-content">•</span> <i>y</i> = 42.9 (±2.83; <i>p</i> &amp;lt; 0.001) + incubation (random, <i>p</i> = 0.34) + 0.035 (±0.0044; <i>p</i> &amp;lt; 0.001) <i>x</i>; <i>R</i> <sup>2</sup> = 0.62; (iii) VFA, gases and AA with minimum incorporation of [2H] into AA synthesis: <span class="styled-content" tagx="styled-content" title="styled-content">•</span> <i>y</i> = 38.1 (±1.89; <i>p</i> &amp;lt; 0.001) + incubation (random, <i>p</i> = 0.60) + 0.038 (±0.0045; <i>p</i> &amp;lt; 0.001) <i>x</i>; <i>R</i> <sup>2</sup> = 0.60. </p>   </div> <div class="fig" title="fig"> <div class="figure" title="figure"> <a href="http://doi.org/"> <span class="label" tagx="label" title="label">Figure 3</span> </a> </div>   <p>Dendrogram of bacterial denaturing gradient gel electrophoresis (DGGE) profile of the rumen in vitro batch cultures. ( <b>a</b>) DGGE bacterial profiles of: BTCM, bromotrichlorometane; CL, chloroform; BES, 2-bromoethanesulfonate; Control, control treatment containing no inhibitor; AQ, 9,10-anthraquinone; ( <b>b</b>) DGGE bacterial profiles of: SN, sodium nitrate; Control, control treatment containing no inhibitor; LO, linseed oil; E2B, ethyl-2-butynoate; PA, propynoic acid. Samples of two replicates per treatment of incubation run three were run on two DGGE gels. </p>   </div> <div class="table-wrap_UNKNOWN">
microorganisms-07-00115-t001_Table 1 <span class="label" tagx="label" title="label">Table 1</span>  <p>Effects of several inhibitors of methanogenesis on fermentation of rumen mixed batch cultures. </p>  <table> <thead> <tr> <th>Response</th> <th>Treatment</th> </tr> <tr> <th>Control</th> <th>BES <sup>1</sup> </th> <th>AQ</th> <th>CL</th> <th>BTCM</th> <th>PA</th> <th>E2B</th> <th>LO</th> <th>SN</th> <th>SEM</th> <th> <i>p</i> = </th> </tr> </thead> <tbody> <tr> <td>Gas production (mmol)</td> <td>1.80 <sup>a3</sup> </td> <td>1.63 <sup>a</sup> </td> <td>0.91 <sup>cd</sup> </td> <td>1.67 <sup>a</sup> </td> <td>1.59 <sup>ab</sup> </td> <td>0.79 <sup>d</sup> </td> <td>1.24 <sup>bc</sup> </td> <td>1.71 <sup>a</sup> </td> <td>0.52 <sup>d</sup> </td> <td>0.17</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>CH <sub>4</sub> <sup>2</sup> (mol/100 mol total gas) </td> <td>10.9 <sup>a</sup> </td> <td>0.55 <sup>bc</sup> </td> <td>2.34 <sup>bc</sup> </td> <td>1.36 <sup>bc</sup> </td> <td>2.70 <sup>bc</sup> </td> <td>0.076 <sup>bc</sup> </td> <td>3.13 <sup>b</sup> </td> <td>8.05 <sup>a</sup> </td> <td>ND <sup>4</sup> </td> <td>1.05</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>CH <sub>4</sub> production (µmol/g incubated OM) </td> <td>242 <sup>a</sup> </td> <td>14.3 <sup>b</sup> </td> <td>29.3 <sup>b</sup> </td> <td>46.7 <sup>b</sup> </td> <td>51.1 <sup>b</sup> </td> <td>1.79 <sup>b</sup> </td> <td>62.1 <sup>b</sup> </td> <td>205 <sup>a</sup> </td> <td>ND</td> <td>28.3</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>CH <sub>4</sub> production (µmol/g truly digested OM) </td> <td>523 <sup>a</sup> </td> <td>30.4 <sup>bc</sup> </td> <td>121 <sup>bc</sup> </td> <td>68.3 <sup>bc</sup> </td> <td>110 <sup>bc</sup> </td> <td>9.26 <sup>c</sup> </td> <td>257 <sup>b</sup> </td> <td>575 <sup>a</sup> </td> <td>ND</td> <td>57.5</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>H <sub>2</sub> accumulation (mol/100 mol total gas) </td> <td>0.91 <sup>cd</sup> </td> <td>2.91 <sup>b</sup> </td> <td>5.60 <sup>a</sup> </td> <td>2.34 <sup>bc</sup> </td> <td>2.42 <sup>bc</sup> </td> <td>1.68 <sup>bcd</sup> </td> <td>0.36 <sup>d</sup> </td> <td>0.84 <sup>cd</sup> </td> <td>0.39 <sup>d</sup> </td> <td>0.64</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>H <sub>2</sub> accumulation (µmol/g incubated OM) </td> <td>14.7 <sup>c</sup> </td> <td>60.2 <sup>a</sup> </td> <td>76.4 <sup>a</sup> </td> <td>51.2 <sup>ab</sup> </td> <td>52.3 <sup>a</sup> </td> <td>21.7 <sup>bc</sup> </td> <td>6.79 <sup>c</sup> </td> <td>19.5 <sup>c</sup> </td> <td>2.97 <sup>c</sup> </td> <td>9.95</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>H <sub>2</sub> accumulation (µmol/g truly digested OM) </td> <td>37.4 <sup>bc</sup> </td> <td>155 <sup>abc</sup> </td> <td>307 <sup>a</sup> </td> <td>213 <sup>abc</sup> </td> <td>257 <sup>ab</sup> </td> <td>84.6 <sup>abc</sup> </td> <td>26.8 <sup>c</sup> </td> <td>68.3 <sup>bc</sup> </td> <td>8.52 <sup>c</sup> </td> <td>65.3</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Final pH</td> <td>6.72 <sup>c</sup> </td> <td>6.70 <sup>c</sup> </td> <td>6.83 <sup>a</sup> </td> <td>6.72 <sup>c</sup> </td> <td>6.72 <sup>c</sup> </td> <td>6.79 <sup>ab</sup> </td> <td>6.76 <sup>bc</sup> </td> <td>6.70 <sup>c</sup> </td> <td>6.84 <sup>a</sup> </td> <td>0.042</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Final E <i>h</i> </td> <td>−163 <sup>ab</sup> </td> <td>−167 <sup>ab</sup> </td> <td>−220 <sup>c</sup> </td> <td>−171 <sup>b</sup> </td> <td>−170 <sup>b</sup> </td> <td>−167 <sup>ab</sup> </td> <td>−165 <sup>ab</sup> </td> <td>−157 <sup>ab</sup> </td> <td>−140 <sup>a</sup> </td> <td>6.06</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Total VFA (mM)</td> <td>27.2 <sup>a</sup> </td> <td>26.1 <sup>ab</sup> </td> <td>20.7 <sup>bc</sup> </td> <td>26.0 <sup>ab</sup> </td> <td>25.5 <sup>ab</sup> </td> <td>17.3 <sup>c</sup> </td> <td>23.7 <sup>ab</sup> </td> <td>27.2 <sup>a</sup> </td> <td>17.7 <sup>c</sup> </td> <td>7.95</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Acetate (mol/100 mol)</td> <td>60.2 <sup>cd</sup> </td> <td>55.3 <sup>d</sup> </td> <td>62.3 <sup>c</sup> </td> <td>56.7 <sup>cd</sup> </td> <td>57.0 <sup>cd</sup> </td> <td>70.0 <sup>a</sup> </td> <td>63.2 <sup>bc</sup> </td> <td>57.8 <sup>cd</sup> </td> <td>69.7 <sup>ab</sup> </td> <td>2.18</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Propionate (mol/100 mol)</td> <td>26.8 <sup>a</sup> </td> <td>30.0 <sup>a</sup> </td> <td>19.3 <sup>b</sup> </td> <td>29.5 <sup>a</sup> </td> <td>29.5 <sup>a</sup> </td> <td>15.8 <sup>b</sup> </td> <td>19.6 <sup>b</sup> </td> <td>29.4 <sup>a</sup> </td> <td>16.8 <sup>b</sup> </td> <td>2.06</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Butyrate (mol/100 mol)</td> <td>8.04 <sup>c</sup> </td> <td>9.65 <sup>bc</sup> </td> <td>11.6 <sup>ab</sup> </td> <td>8.63 <sup>c</sup> </td> <td>8.55 <sup>c</sup> </td> <td>7.56 <sup>c</sup> </td> <td>12.7 <sup>a</sup> </td> <td>7.92 <sup>c</sup> </td> <td>7.13 <sup>c</sup> </td> <td>1.06</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Isobutyrate (mol/100 mol)</td> <td>1.37</td> <td>1.48</td> <td>1.73</td> <td>1.67</td> <td>1.49</td> <td>1.70</td> <td>1.66</td> <td>1.47</td> <td>1.68</td> <td>0.32</td> <td>0.11</td> </tr> <tr> <td>2- and 3-methylbutyrate (mol/100 mol)</td> <td>1.86</td> <td>1.75</td> <td>2.31</td> <td>1.77</td> <td>1.63</td> <td>1.88</td> <td>2.53</td> <td>1.88</td> <td>1.92</td> <td>1.02</td> <td>0.081</td> </tr> <tr> <td>Valerate (mol/100 mol)</td> <td>1.30 <sup>c</sup> </td> <td>1.24 <sup>c</sup> </td> <td>1.80 <sup>a</sup> </td> <td>1.22 <sup>c</sup> </td> <td>1.32 <sup>c</sup> </td> <td>1.43 <sup>bc</sup> </td> <td>1.69 <sup>ab</sup> </td> <td>1.19 <sup>c</sup> </td> <td>1.78 <sup>a</sup> </td> <td>0.22</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Caproate (mol/100 mol)</td> <td>0.44 <sup>b</sup> </td> <td>0.56 <sup>b</sup> </td> <td>0.88 <sup>ab</sup> </td> <td>0.50 <sup>b</sup> </td> <td>0.50 <sup>b</sup> </td> <td>1.75 <sup>a</sup> </td> <td>0.42 <sup>b</sup> </td> <td>0.43 <sup>b</sup> </td> <td>1.00 <sup>ab</sup> </td> <td>0.44</td> <td>0.008</td> </tr> <tr> <td>Acetate/propionate (mol/mol)</td> <td>2.36 <sup>bcd</sup> </td> <td>1.96 <sup>d</sup> </td> <td>3.27 <sup>abc</sup> </td> <td>2.02 <sup>d</sup> </td> <td>2.04 <sup>cd</sup> </td> <td>4.49 <sup>a</sup> </td> <td>3.57 <sup>ab</sup> </td> <td>2.07 <sup>cd</sup> </td> <td>4.30 <sup>a</sup> </td> <td>0.30</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>NH <sub>4</sub> <sup>+</sup> (mM) </td> <td>8.26</td> <td>8.77</td> <td>9.29</td> <td>8.64</td> <td>8.95</td> <td>9.09</td> <td>9.09</td> <td>8.87</td> <td>9.24</td> <td>1.00</td> <td>0.096</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <sup>1</sup> BES, 2-bromoethanesulfonate; AQ, 9, 10-anthraquinone; CL, chloroform; BTCM, bromotrichlorometane; PA, propynoic acid; E2B, ethyl-2-butynoate; LO, linseed oil; SN, sodium nitrate. <sup>2</sup> CH <sub>4</sub>, methane; H <sub>2</sub>, dihydrogen; E <i>h</i>, reducing potential; VFA, volatile fatty acids; NH <sub>4</sub> <sup>+</sup>, ammonium. <sup>3</sup> Unlike superscripts within the same row indicate significant ( <i>p</i> &amp;lt; 0.05) differences according to Tukey HSD. <sup>4</sup> ND, not detected. </p> </div> </div> </div> <div class="table-wrap_UNKNOWN">
microorganisms-07-00115-t002_Table 2 <span class="label" tagx="label" title="label">Table 2</span>  <p>Effects of several inhibitors of methanogenesis on digestion and microbial biomass production and composition of mixed rumen batch cultures. </p>  <table> <thead> <tr> <th>Response</th> <th>Treatment</th> <th>SEM</th> <th> <i>p</i> = </th> </tr> <tr> <th>Control</th> <th>BES <sup>1</sup> </th> <th>AQ</th> <th>CL</th> <th>BTCM</th> <th>PA</th> <th>E2B</th> <th>LO</th> <th>SN</th> </tr> </thead> <tbody> <tr> <td>True OM digestibility (%) <sup>2</sup> </td> <td>42.7 <sup>a 3</sup> </td> <td>41.4 <sup>ab</sup> </td> <td>31.2 <sup>ab</sup> </td> <td>46.4 <sup>a</sup> </td> <td>34.2 <sup>ab</sup> </td> <td>27.2 <sup>ab</sup> </td> <td>22.8 <sup>b</sup> </td> <td>31.4 <sup>ab</sup> </td> <td>31.2 <sup>ab</sup> </td> <td>6.13</td> <td>0.008</td> </tr> <tr> <td>Microbial OM (mg)</td> <td>170</td> <td>215</td> <td>223</td> <td>164</td> <td>155</td> <td>209</td> <td>136</td> <td>120</td> <td>188</td> <td>28.7</td> <td>0.14</td> </tr> <tr> <td>Microbial N (mg)</td> <td>3.13 <sup>ab</sup> </td> <td>2.12 <sup>bc</sup> </td> <td>1.75 <sup>bc</sup> </td> <td>3.20 <sup>ab</sup> </td> <td>3.55 <sup>ab</sup> </td> <td>1.19 <sup>c</sup> </td> <td>1.97 <sup>bc</sup> </td> <td>3.97 <sup>a</sup> </td> <td>1.37 <sup>c</sup> </td> <td>0.47</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Total microbial AA-N (mg)</td> <td>1.48</td> <td>1.20</td> <td>0.87</td> <td>1.51</td> <td>1.56</td> <td>0.94</td> <td>0.89</td> <td>1.59</td> <td>1.48</td> <td>0.16</td> <td>0.001</td> </tr> <tr> <td>Amino acid (g/100 g total AA)</td> </tr> <tr> <td>Asp</td> <td>11.7 <sup>bc</sup> </td> <td>12.8 <sup>ab</sup> </td> <td>5.73 <sup>c</sup> </td> <td>11.9 <sup>abc</sup> </td> <td>12.0 <sup>abc</sup> </td> <td>9.43 <sup>bc</sup> </td> <td>13.8 <sup>ab</sup> </td> <td>13.3 <sup>ab</sup> </td> <td>16.9 <sup>a</sup> </td> <td>1.23</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Glu</td> <td>18.1</td> <td>10.9</td> <td>15.0</td> <td>19.8</td> <td>19.9</td> <td>18.0</td> <td>11.8</td> <td>18.0</td> <td>28.5</td> <td>3.95</td> <td>0.067</td> </tr> <tr> <td>Ser</td> <td>5.28 <sup>a</sup> </td> <td>5.55 <sup>a</sup> </td> <td>4.85 <sup>abc</sup> </td> <td>4.81 <sup>abc</sup> </td> <td>5.14 <sup>ab</sup> </td> <td>3.34 <sup>c</sup> </td> <td>6.12 <sup>a</sup> </td> <td>5.23 <sup>a</sup> </td> <td>3.53 <sup>bc</sup> </td> <td>0.42</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Gly</td> <td>6.27 <sup>ab</sup> </td> <td>6.67 <sup>ab</sup> </td> <td>6.33 <sup>ab</sup> </td> <td>5.32 <sup>b</sup> </td> <td>6.14 <sup>ab</sup> </td> <td>7.17 <sup>ab</sup> </td> <td>8.11 <sup>a</sup> </td> <td>6.32 <sup>ab</sup> </td> <td>6.14 <sup>ab</sup> </td> <td>0.70</td> <td>0.008</td> </tr> <tr> <td>His</td> <td>1.18 <sup>ab</sup> </td> <td>2.25 <sup>ab</sup> </td> <td>2.21 <sup>ab</sup> </td> <td>1.71 <sup>ab</sup> </td> <td>2.38 <sup>a</sup> </td> <td>0.62 <sup>b</sup> </td> <td>0.52 <sup>b</sup> </td> <td>1.47 <sup>ab</sup> </td> <td>0.73 <sup>ab</sup> </td> <td>0.47</td> <td>0.005</td> </tr> <tr> <td>Arg</td> <td>4.26 <sup>ab</sup> </td> <td>4.98 <sup>a</sup> </td> <td>4.41 <sup>ab</sup> </td> <td>3.38 <sup>bcd</sup> </td> <td>4.26 <sup>abc</sup> </td> <td>2.82 <sup>cd</sup> </td> <td>3.23 <sup>bcd</sup> </td> <td>3.65 <sup>abc</sup> </td> <td>2.03 <sup>d</sup> </td> <td>0.48</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Thr</td> <td>5.50 <sup>a</sup> </td> <td>5.77 <sup>a</sup> </td> <td>5.33 <sup>a</sup> </td> <td>4.97 <sup>a</sup> </td> <td>5.05 <sup>a</sup> </td> <td>2.86 <sup>b</sup> </td> <td>5.46 <sup>a</sup> </td> <td>4.94 <sup>a</sup> </td> <td>2.37 <sup>b</sup> </td> <td>0.43</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Ala</td> <td>8.58 <sup>b</sup> </td> <td>10.2 <sup>ab</sup> </td> <td>13.8 <sup>a</sup> </td> <td>8.34 <sup>b</sup> </td> <td>8.42 <sup>b</sup> </td> <td>10.8 <sup>ab</sup> </td> <td>14.2 <sup>a</sup> </td> <td>10.1 <sup>ab</sup> </td> <td>8.93 <sup>b</sup> </td> <td>1.67</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Pro</td> <td>3.26 <sup>bc</sup> </td> <td>3.83 <sup>abc</sup> </td> <td>4.46 <sup>ab</sup> </td> <td>3.93 <sup>abc</sup> </td> <td>3.61 <sup>abc</sup> </td> <td>4.74 <sup>a</sup> </td> <td>3.57 <sup>abc</sup> </td> <td>3.39 <sup>abc</sup> </td> <td>2.78 <sup>c</sup> </td> <td>0.36</td> <td>0.002</td> </tr> <tr> <td>Tyr</td> <td>4.42 <sup>a</sup> </td> <td>4.57 <sup>a</sup> </td> <td>3.93 <sup>ab</sup> </td> <td>4.57 <sup>a</sup> </td> <td>3.75 <sup>ab</sup> </td> <td>1.83 <sup>b</sup> </td> <td>3.13 <sup>ab</sup> </td> <td>3.91 <sup>ab</sup> </td> <td>2.01 <sup>b</sup> </td> <td>0.60</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Val</td> <td>6.03 <sup>a</sup> </td> <td>7.42 <sup>a</sup> </td> <td>8.13 <sup>a</sup> </td> <td>5.65 <sup>a</sup> </td> <td>6.09 <sup>a</sup> </td> <td>6.50 <sup>a</sup> </td> <td>7.33 <sup>a</sup> </td> <td>5.58 <sup>a</sup> </td> <td>5.82 <sup>a</sup> </td> <td>0.63</td> <td>0.032</td> </tr> <tr> <td>Ile</td> <td>5.50</td> <td>6.45</td> <td>6.75</td> <td>5.01</td> <td>5.35</td> <td>5.95</td> <td>5.94</td> <td>6.10</td> <td>4.89</td> <td>0.41</td> <td>0.058</td> </tr> <tr> <td>Leu</td> <td>7.02 <sup>a</sup> </td> <td>7.95 <sup>a</sup> </td> <td>7.55 <sup>a</sup> </td> <td>6.14 <sup>ab</sup> </td> <td>6.54 <sup>a</sup> </td> <td>3.28 <sup>c</sup> </td> <td>7.75 <sup>a</sup> </td> <td>6.77 <sup>a</sup> </td> <td>3.95 <sup>bc</sup> </td> <td>0.66</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Lys</td> <td>7.61</td> <td>8.81</td> <td>9.35</td> <td>7.00</td> <td>7.32</td> <td>7.40</td> <td>7.77</td> <td>8.54</td> <td>7.78</td> <td>0.61</td> <td>0.17</td> </tr> <tr> <td>Phe</td> <td>4.49 <sup>a</sup> </td> <td>4.71 <sup>a</sup> </td> <td>4.35 <sup>ab</sup> </td> <td>3.80 <sup>ab</sup> </td> <td>3.92 <sup>ab</sup> </td> <td>3.55 <sup>ab</sup> </td> <td>2.98 <sup>b</sup> </td> <td>3.98 <sup>ab</sup> </td> <td>3.57 <sup>ab</sup> </td> <td>0.37</td> <td>0.009</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <sup>1</sup> BES, 2-bromoethanesulfonate; AQ, 9, 10-anthraquinone; CL, chloroform; BTCM, bromotrichlorometane; PA, propynoic acid; E2B, ethyl-2-butynoate; LO, linseed oil; SN, sodium nitrate; <sup>2</sup> OM, organic matter; N, nitrogen; AA, amino acids. <sup>3</sup> Unlike superscripts within the same row indicate significant ( <i>p</i> &amp;lt; 0.05) differences according to Tukey HSD. </p> </div> </div> </div> <div class="table-wrap_UNKNOWN">
microorganisms-07-00115-t003_Table 3 <span class="label" tagx="label" title="label">Table 3</span>  <p>Effects of several inhibitors of methanogenesis on the balance of reducing equivalents pairs ([2H]) in ruminal batch cultures. </p>  <table> <thead> <tr> <th>Response</th> <th>Treatment</th> <th>SEM</th> <th> <i>p</i> = </th> </tr> <tr> <th>Control</th> <th>BES <sup>1</sup> </th> <th>AQ</th> <th>CL</th> <th>BTCM</th> <th>PA</th> <th>E2B</th> <th>LO</th> <th>SN</th> </tr> </thead> <tbody> <tr> <td> <b>VFA and Gases</b> </td> </tr> <tr> <td> <b>[2H] Produced</b> </td> </tr> <tr> <td>Acetate (mmol)</td> <td>4.17 <sup>a3</sup> </td> <td>3.79 <sup>ab</sup> </td> <td>3.33 <sup>bc</sup> </td> <td>3.81 <sup>ab</sup> </td> <td>3.76 <sup>ab</sup> </td> <td>2.99 <sup>c</sup> </td> <td>4.38 <sup>a</sup> </td> <td>4.14 <sup>a</sup> </td> <td>3.00 <sup>c</sup> </td> <td>1.27</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Propionate (mmol)</td> <td>0.91 <sup>a</sup> </td> <td>0.97 <sup>a</sup> </td> <td>0.53 <sup>b</sup> </td> <td>0.95 <sup>a</sup> </td> <td>0.93 <sup>a</sup> </td> <td>0.38 <sup>b</sup> </td> <td>0.56 <sup>b</sup> </td> <td>1.02 <sup>a</sup> </td> <td>0.41 <sup>b</sup> </td> <td>0.21</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Butyrate (mmol)</td> <td>1.28 <sup>b</sup> </td> <td>1.39 <sup>b</sup> </td> <td>1.26 <sup>bc</sup> </td> <td>1.27 <sup>b</sup> </td> <td>1.25 <sup>b</sup> </td> <td>0.81 <sup>c</sup> </td> <td>2.01 <sup>a</sup> </td> <td>1.35 <sup>b</sup> </td> <td>0.83 <sup>c</sup> </td> <td>0.53</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Valerate (mmol)</td> <td>0.16 <sup>ab</sup> </td> <td>0.15 <sup>abc</sup> </td> <td>0.15 <sup>abc</sup> </td> <td>0.14 <sup>abc</sup> </td> <td>0.14 <sup>abc</sup> </td> <td>0.11 <sup>c</sup> </td> <td>0.18 <sup>a</sup> </td> <td>0.14 <sup>abc</sup> </td> <td>0.13 <sup>bc</sup> </td> <td>0.073</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Total [2H] produced (mmol) <sup>2</sup> </td> <td>6.52 <sup>ab</sup> </td> <td>6.30 <sup>ab</sup> </td> <td>5.27 <sup>bc</sup> </td> <td>6.17 <sup>ab</sup> </td> <td>6.08 <sup>ab</sup> </td> <td>4.28 <sup>c</sup> </td> <td>7.13 <sup>a</sup> </td> <td>6.66 <sup>ab</sup> </td> <td>4.38 <sup>c</sup> </td> <td>2.06</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>[2H] incorporated</td> </tr> <tr> <td>Propionate (mmol)</td> <td>1.81 <sup>a</sup> </td> <td>1.94 <sup>a</sup> </td> <td>1.07 <sup>b</sup> </td> <td>1.89 <sup>a</sup> </td> <td>1.86 <sup>a</sup> </td> <td>0.76 <sup>b</sup> </td> <td>1.12 <sup>b</sup> </td> <td>2.04 <sup>a</sup> </td> <td>0.82 <sup>b</sup> </td> <td>0.43</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Butyrate (mmol)</td> <td>0.64 <sup>b</sup> </td> <td>0.70 <sup>b</sup> </td> <td>0.63 <sup>bc</sup> </td> <td>0.63 <sup>b</sup> </td> <td>0.62 <sup>b</sup> </td> <td>0.41 <sup>c</sup> </td> <td>1.01 <sup>a</sup> </td> <td>0.68 <sup>b</sup> </td> <td>0.42 <sup>c</sup> </td> <td>0.26</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Valerate (mmol)</td> <td>0.22 <sup>ab</sup> </td> <td>0.20 <sup>abc</sup> </td> <td>0.20 <sup>abc</sup> </td> <td>0.19 <sup>abc</sup> </td> <td>0.18 <sup>abc</sup> </td> <td>0.14 <sup>c</sup> </td> <td>0.24 <sup>a</sup> </td> <td>0.19 <sup>abc</sup> </td> <td>0.18 <sup>bc</sup> </td> <td>0.097</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>CH <sub>4</sub> (mmol) </td> <td>0.78 <sup>a</sup> </td> <td>0.037 <sup>b</sup> </td> <td>0.090 <sup>b</sup> </td> <td>0.15 <sup>b</sup> </td> <td>0.25 <sup>b</sup> </td> <td>0.0060 <sup>b</sup> </td> <td>0.26 <sup>b</sup> </td> <td>0.70 <sup>a</sup> </td> <td>ND <sup>3</sup> </td> <td>0.11</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>H <sub>2</sub> (mmol) </td> <td>0.012 <sup>c</sup> </td> <td>0.046 <sup>a</sup> </td> <td>0.048 <sup>a</sup> </td> <td>0.041 <sup>a</sup> </td> <td>0.039 <sup>ab</sup> </td> <td>0.017 <sup>bc</sup> </td> <td>0.0082 <sup>c</sup> </td> <td>0.016 <sup>bc</sup> </td> <td>0.0024 <sup>c</sup> </td> <td>0.0082</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Total [2H] incorporated (mmol)</td> <td>3.45 <sup>a</sup> </td> <td>2.91 <sup>ab</sup> </td> <td>2.09 <sup>bc</sup> </td> <td>2.85 <sup>ab</sup> </td> <td>2.97 <sup>ab</sup> </td> <td>1.33 <sup>c</sup> </td> <td>2.62 <sup>ab</sup> </td> <td>3.62 <sup>a</sup> </td> <td>1.35 <sup>c</sup> </td> <td>0.76</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>[2H] recovery (%)</td> <td>59.2 <sup>a</sup> </td> <td>48.9 <sup>ab</sup> </td> <td>40.1 <sup>bc</sup> </td> <td>47.1 <sup>ab</sup> </td> <td>49.5 <sup>ab</sup> </td> <td>29.2 <sup>c</sup> </td> <td>37.4 <sup>bc</sup> </td> <td>60.8 <sup>a</sup> </td> <td>27.1 <sup>c4</sup> </td> <td>3.72</td> <td>&amp;lt;0.001</td> </tr> <tr> <td> <b>Amino acids</b> </td> </tr> <tr> <td>Max [2H] net incorporation (mmol)</td> <td>0.10</td> <td>0.098</td> <td>0.072</td> <td>0.13</td> <td>0.13</td> <td>0.084</td> <td>0.073</td> <td>0.12</td> <td>0.13</td> <td>0.022</td> <td>0.34</td> </tr> <tr> <td>Min [2H] net incorporation (mmol)</td> <td>−0.11</td> <td>−0.097</td> <td>−0.060</td> <td>−0.13</td> <td>−0.14</td> <td>−0.091</td> <td>−0.060</td> <td>−0.13</td> <td>−0.15</td> <td>0.024</td> <td>0.16</td> </tr> <tr> <td> <b>Overall (VFA + gases + AA)</b> </td> </tr> <tr> <td>Max [2H] recovery (%)</td> <td>62.0 <sup>a</sup> </td> <td>52.1 <sup>ab</sup> </td> <td>41.9 <sup>abc</sup> </td> <td>52.2 <sup>ab</sup> </td> <td>55.8 <sup>ab</sup> </td> <td>35.6 <sup>c</sup> </td> <td>38.1 <sup>bc</sup> </td> <td>63.3 <sup>a</sup> </td> <td>37.4 <sup>c4</sup> </td> <td>4.90</td> <td>&amp;lt;0.001</td> </tr> <tr> <td>Min [2H] recovery (%)</td> <td>58.7 <sup>a</sup> </td> <td>49.1 <sup>ab</sup> </td> <td>40.1 <sup>abc</sup> </td> <td>47.5 <sup>ab</sup> </td> <td>51.3 <sup>ab</sup> </td> <td>31.0 <sup>c</sup> </td> <td>36.9 <sup>bc</sup> </td> <td>59.1 <sup>a</sup> </td> <td>29.3 <sup>c4</sup> </td> <td>4.13</td> <td>&amp;lt;0.001</td> </tr> </tbody> </table> <div class="table-wrap-foot_UNKNOWN"> <div class="fn-type-" title=""> <p> <sup>1</sup> BES, 2-bromoethanesulfonate; AQ, 9, 10-anthraquinone; CL, chloroform; BTCM, bromotrichlorometane; PA, propynoic acid; E2B, ethyl-2-butynoate; LO, linseed oil; SN, sodium nitrate. <sup>2</sup> [2H], reducing equivalents pairs, CH <sub>4</sub>, methane; H <sub>2</sub>, dihydrogen; VFA, volatile fatty acids; AA, amino acids. <sup>3</sup> Unlike superscripts within the same row indicate significant ( <i>p</i> &amp;lt; 0.05) differences according to Tukey HSD. <sup>3</sup> ND, not detected. <sup>4</sup> Probably underestimated as the reduction of nitrate to nitrite and ammonium was not determined. </p> </div> </div> </div> </div>  </body></html>